diff --git a/extensions/browser/updater/update_service.cc b/extensions/browser/updater/update_service.cc
index 29ff34fa0a..23baad2e5a 100644
--- a/extensions/browser/updater/update_service.cc
+++ b/extensions/browser/updater/update_service.cc
@@ -132,13 +132,6 @@ void UpdateService::OnCrxStateChange(UpdateFoundCallback update_found_callback,
       break;
   }
 
-  if (should_perform_action_on_omaha_attributes) {
-    base::Value::Dict attributes = GetExtensionOmahaAttributes(item);
-    // Note that it's important to perform actions even if |attributes| is
-    // empty, missing values may default to false and have associated logic.
-    ExtensionSystem::Get(browser_context_)
-        ->PerformActionBasedOnOmahaAttributes(item.id, attributes);
-  }
 }
 
 UpdateService::UpdateService(
