diff --git a/chrome/browser/ui/startup/first_run_service.cc b/chrome/browser/ui/startup/first_run_service.cc
index f9289f66a1..0db2371a4d 100644
--- a/chrome/browser/ui/startup/first_run_service.cc
+++ b/chrome/browser/ui/startup/first_run_service.cc
@@ -28,7 +28,6 @@
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/consent_level.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/account_info.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
@@ -74,32 +73,7 @@ enum class PolicyEffect {
 };
 
 PolicyEffect ComputeDevicePolicyEffect(Profile& profile) {
-  const PrefService* const local_state = g_browser_process->local_state();
-  if (!local_state->GetBoolean(prefs::kPromotionsEnabled)) {
-    // Corresponding policy: PromotionsEnabled=false
-    return PolicyEffect::kDisabled;
-  }
-
-  if (!SyncServiceFactory::IsSyncAllowed(&profile)) {
-    // Corresponding policy: SyncDisabled=true
-    return PolicyEffect::kDisabled;
-  }
-
-  if (signin_util::IsForceSigninEnabled()) {
-    // Corresponding policy: BrowserSignin=2
-    // Debugging note: On Linux this policy is not supported and does not get
-    // translated to the prefs (see crbug.com/956998), but we still respond to
-    // `prefs::kForceBrowserSignin` being set (e.g. if manually edited).
-    return PolicyEffect::kDisabled;
-  }
-
-  if (!profile.GetPrefs()->GetBoolean(prefs::kSigninAllowed) ||
-      !profile.GetPrefs()->GetBoolean(prefs::kSigninAllowedOnNextStartup)) {
-    // Corresponding policy: BrowserSignin=0
-    return PolicyEffect::kDisabled;
-  }
-
-  return PolicyEffect::kNone;
+  return PolicyEffect::kDisabled;
 }
 
 void SetFirstRunFinished(FirstRunService::FinishedReason reason) {
