diff --git a/components/search_engines/template_url.cc b/components/search_engines/template_url.cc
index 86474ebbec..2e7838e8b5 100644
--- a/components/search_engines/template_url.cc
+++ b/components/search_engines/template_url.cc
@@ -609,10 +609,7 @@ std::u16string TemplateURLRef::SearchTermToString16(
 bool TemplateURLRef::HasGoogleBaseURLs(
     const SearchTermsData& search_terms_data) const {
   ParseIfNecessary(search_terms_data);
-  return std::ranges::any_of(replacements_, [](const Replacement& replacement) {
-    return replacement.type == GOOGLE_BASE_URL ||
-           replacement.type == GOOGLE_BASE_SUGGEST_URL;
-  });
+  return false;
 }
 
 bool TemplateURLRef::ExtractSearchTermsFromURL(
