diff --git a/components/search_engines/template_url_service.h b/components/search_engines/template_url_service.h
index a8152c71e4..6df589b4e1 100644
--- a/components/search_engines/template_url_service.h
+++ b/components/search_engines/template_url_service.h
@@ -345,7 +345,8 @@ class TemplateURLService final : public WebDataServiceConsumer,
   void ResetTemplateURL(TemplateURL* url,
                         const std::u16string& title,
                         const std::u16string& keyword,
-                        const std::string& search_url);
+                        const std::string& search_url,
+                        const std::string& suggestions_url);
 
   // Sets the `is_active` field of the specified TemplateURL to `kTrue` or
   // `kFalse`. Called when a user explicitly activates/deactivates the search
