diff --git a/chrome/browser/ui/webui/searchbox/searchbox_omnibox_client.cc b/chrome/browser/ui/webui/searchbox/searchbox_omnibox_client.cc
index bfcd1d14d2..550642b80d 100644
--- a/chrome/browser/ui/webui/searchbox/searchbox_omnibox_client.cc
+++ b/chrome/browser/ui/webui/searchbox/searchbox_omnibox_client.cc
@@ -156,14 +156,6 @@ void SearchboxOmniboxClient::OnAutocompleteAccept(
                              transition, false),
       /*navigation_handle_callback=*/{});
 
-#if BUILDFLAG(ENABLE_EXTENSIONS)
-  if (AutocompleteMatch::IsSearchType(match.type)) {
-    if (auto* telemetry_service =
-            safe_browsing::ExtensionTelemetryService::Get(profile_)) {
-      telemetry_service->OnOmniboxSearch(match);
-    }
-  }
-#endif
 }
 
 base::WeakPtr<OmniboxClient> SearchboxOmniboxClient::AsWeakPtr() {
