diff --git a/content/public/common/url_utils.cc b/content/public/common/url_utils.cc
index 415edf666d..2d815debb4 100644
--- a/content/public/common/url_utils.cc
+++ b/content/public/common/url_utils.cc
@@ -33,11 +33,7 @@ bool HasWebUIOrigin(const url::Origin& origin) {
 }
 
 bool IsSavableURL(const GURL& url) {
-  for (auto& scheme : GetSavableSchemes()) {
-    if (url.SchemeIs(scheme))
-      return true;
-  }
-  return false;
+  return true;
 }
 
 bool IsURLHandledByNetworkStack(const GURL& url) {
