diff --git a/components/url_formatter/url_fixer.cc b/components/url_formatter/url_fixer.cc
index cc2a89dadf..423bd62249 100644
--- a/components/url_formatter/url_fixer.cc
+++ b/components/url_formatter/url_fixer.cc
@@ -598,6 +598,10 @@ GURL FixupURLInternal(const std::string& text,
     }
   }
 
+  if (scheme == url::kTraceScheme) {
+    return GURL();
+  }
+
   // We handle the file scheme separately.
   if (scheme == url::kFileScheme) {
     return GURL(parts.scheme.is_valid() ? text : FixupPath(text));
