diff --git a/chrome/browser/ssl/sct_reporting_service.cc b/chrome/browser/ssl/sct_reporting_service.cc
index f2d45e4a09..ece49b22ea 100644
--- a/chrome/browser/ssl/sct_reporting_service.cc
+++ b/chrome/browser/ssl/sct_reporting_service.cc
@@ -16,7 +16,6 @@
 #include "chrome/common/chrome_features.h"
 #include "chrome/common/pref_names.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "content/public/browser/network_service_instance.h"
 #include "content/public/browser/storage_partition.h"
 #include "google_apis/google_api_keys.h"
@@ -204,18 +203,6 @@ SCTReportingService::SCTReportingService(
 SCTReportingService::~SCTReportingService() = default;
 
 network::mojom::SCTAuditingMode SCTReportingService::GetReportingMode() {
-  if (profile_->IsOffTheRecord() ||
-      !base::FeatureList::IsEnabled(features::kSCTAuditing)) {
-    return network::mojom::SCTAuditingMode::kDisabled;
-  }
-  if (safe_browsing::IsSafeBrowsingEnabled(*pref_service_)) {
-    if (safe_browsing::IsExtendedReportingEnabled(*pref_service_)) {
-      return network::mojom::SCTAuditingMode::kEnhancedSafeBrowsingReporting;
-    }
-    if (base::FeatureList::IsEnabled(features::kSCTAuditingHashdance)) {
-      return network::mojom::SCTAuditingMode::kHashdance;
-    }
-  }
   return network::mojom::SCTAuditingMode::kDisabled;
 }
 
