diff --git a/components/safe_browsing/core/browser/tailored_security_service/tailored_security_service.cc b/components/safe_browsing/core/browser/tailored_security_service/tailored_security_service.cc
index 3de9d320ee..6fe7eef42b 100644
--- a/components/safe_browsing/core/browser/tailored_security_service/tailored_security_service.cc
+++ b/components/safe_browsing/core/browser/tailored_security_service/tailored_security_service.cc
@@ -24,7 +24,6 @@
 #include "components/safe_browsing/core/browser/tailored_security_service/tailored_security_service_util.h"
 #include "components/safe_browsing/core/common/features.h"
 #include "components/safe_browsing/core/common/safe_browsing_policy_handler.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/common/utils.h"
 #include "components/signin/public/identity_manager/access_token_info.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
@@ -239,15 +238,6 @@ TailoredSecurityService::TailoredSecurityService(
     : identity_manager_(identity_manager),
       sync_service_(sync_service),
       prefs_(prefs) {
-  // `prefs` can be nullptr in unit tests.
-  if (prefs_) {
-    pref_registrar_.Init(prefs_);
-    pref_registrar_.Add(
-        prefs::kAccountTailoredSecurityUpdateTimestamp,
-        base::BindRepeating(
-            &TailoredSecurityService::TailoredSecurityTimestampUpdateCallback,
-            weak_ptr_factory_.GetWeakPtr()));
-  }
 }
 
 TailoredSecurityService::~TailoredSecurityService() {
@@ -405,7 +395,6 @@ void TailoredSecurityService::MaybeNotifySyncUser(bool is_enabled,
       RecordEnabledNotificationResult(
           TailoredSecurityNotificationResult::kHistoryNotSynced);
     }
-    SaveRetryState(TailoredSecurityRetryState::NO_RETRY_NEEDED);
     return;
   }
 
@@ -415,30 +404,9 @@ void TailoredSecurityService::MaybeNotifySyncUser(bool is_enabled,
       RecordEnabledNotificationResult(
           TailoredSecurityNotificationResult::kSafeBrowsingControlledByPolicy);
     }
-    SaveRetryState(TailoredSecurityRetryState::NO_RETRY_NEEDED);
-    return;
-  }
-
-  if (is_enabled && IsEnhancedProtectionEnabled(*prefs())) {
-    RecordEnabledNotificationResult(
-        TailoredSecurityNotificationResult::kEnhancedProtectionAlreadyEnabled);
-    SaveRetryState(TailoredSecurityRetryState::NO_RETRY_NEEDED);
     return;
   }
 
-  if (is_enabled && !IsEnhancedProtectionEnabled(*prefs())) {
-    for (auto& observer : observer_list_) {
-      observer.OnSyncNotificationMessageRequest(true);
-    }
-  }
-
-  if (!is_enabled && IsEnhancedProtectionEnabled(*prefs()) &&
-      prefs()->GetBoolean(
-          prefs::kEnhancedProtectionEnabledViaTailoredSecurity)) {
-    for (auto& observer : observer_list_) {
-      observer.OnSyncNotificationMessageRequest(false);
-    }
-  }
 }
 
 bool TailoredSecurityService::HistorySyncEnabledForUser() {
@@ -519,25 +487,10 @@ void TailoredSecurityService::Shutdown() {
 }
 
 void TailoredSecurityService::TailoredSecurityTimestampUpdateCallback() {
-  // TODO(crbug.com/40925236): remove sync flow last user interaction pref.
-  prefs_->SetInteger(prefs::kTailoredSecuritySyncFlowLastUserInteractionState,
-                     TailoredSecurityRetryState::UNKNOWN);
-  prefs_->SetTime(prefs::kTailoredSecuritySyncFlowLastRunTime,
-                  base::Time::Now());
-  // If this method fails, then a retry is needed. If it succeeds, the
-  // ChromeTailoredSecurityService will set this value to NO_RETRY_NEEDED for
-  // us.
-  prefs_->SetInteger(prefs::kTailoredSecuritySyncFlowRetryState,
-                     TailoredSecurityRetryState::RETRY_NEEDED);
-
   StartRequest(base::BindOnce(&TailoredSecurityService::MaybeNotifySyncUser,
                               weak_ptr_factory_.GetWeakPtr()));
 }
 
-void TailoredSecurityService::SaveRetryState(TailoredSecurityRetryState state) {
-  prefs_->SetInteger(prefs::kTailoredSecuritySyncFlowRetryState, state);
-}
-
 void TailoredSecurityService::SetCanQuery(bool can_query) {
   can_query_ = can_query;
   if (can_query) {
