diff --git a/components/browser_sync/BUILD.gn b/components/browser_sync/BUILD.gn
index 8c71e9194e..d5383354f3 100644
--- a/components/browser_sync/BUILD.gn
+++ b/components/browser_sync/BUILD.gn
@@ -17,8 +17,6 @@ static_library("browser_sync") {
     "sync_engine_factory_impl.h",
     "sync_internals_message_handler.cc",
     "sync_internals_message_handler.h",
-    "sync_to_signin_migration.cc",
-    "sync_to_signin_migration.h",
   ]
 
   public_deps = [
