diff --git a/components/component_updater/installer_policies/BUILD.gn b/components/component_updater/installer_policies/BUILD.gn
index 46b25c2080..f17605354a 100644
--- a/components/component_updater/installer_policies/BUILD.gn
+++ b/components/component_updater/installer_policies/BUILD.gn
@@ -53,6 +53,7 @@ static_library("installer_policies_no_content_deps") {
     "//components/update_client",
     "//mojo/public/cpp/base:protobuf_support",
     "//services/network/public/cpp",
+    "//third_party/re2",
   ]
 
   # Disallow depending on content.
