diff --git a/chrome/browser/enterprise/signals/profile_signals_collector.cc b/chrome/browser/enterprise/signals/profile_signals_collector.cc
index 7303b55773..ebe0e8ee18 100644
--- a/chrome/browser/enterprise/signals/profile_signals_collector.cc
+++ b/chrome/browser/enterprise/signals/profile_signals_collector.cc
@@ -45,12 +45,8 @@ ProfileSignalsCollector::ProfileSignalsCollector(Profile* profile)
           PolicyBlocklistFactory::GetForBrowserContext(profile)),
       profile_prefs_(profile->GetPrefs()),
       policy_manager_(profile->GetCloudPolicyManager()),
-      connectors_service_(
-          enterprise_connectors::ConnectorsServiceFactory::GetForBrowserContext(
-              profile)),
       profile_id_service_(
           enterprise::ProfileIdServiceFactory::GetForProfile(profile)) {
-  CHECK(connectors_service_);
   CHECK(policy_blocklist_service_);
   CHECK(profile_id_service_);
 }
@@ -68,12 +64,8 @@ void ProfileSignalsCollector::GetProfileSignals(
   signal_response.chrome_remote_desktop_app_blocked =
       device_signals::GetChromeRemoteDesktopAppBlocked(
           policy_blocklist_service_);
-  signal_response.password_protection_warning_trigger =
-      device_signals::GetPasswordProtectionWarningTrigger(profile_prefs_);
   signal_response.profile_enrollment_domain =
       device_signals::TryGetEnrollmentDomain(policy_manager_);
-  signal_response.safe_browsing_protection_level =
-      device_signals::GetSafeBrowsingProtectionLevel(profile_prefs_);
   signal_response.site_isolation_enabled =
       device_signals::GetSiteIsolationEnabled();
   signal_response.profile_id = profile_id_service_->GetProfileId();
