diff --git a/content/services/auction_worklet/private_aggregation_bindings.cc b/content/services/auction_worklet/private_aggregation_bindings.cc
index a82c6a6edd..c7a2daf215 100644
--- a/content/services/auction_worklet/private_aggregation_bindings.cc
+++ b/content/services/auction_worklet/private_aggregation_bindings.cc
@@ -406,53 +406,6 @@ PrivateAggregationBindings::~PrivateAggregationBindings() = default;
 
 void PrivateAggregationBindings::AttachToContext(
     v8::Local<v8::Context> context) {
-  if (!base::FeatureList::IsEnabled(blink::features::kPrivateAggregationApi) ||
-      !blink::features::kPrivateAggregationApiEnabledInProtectedAudience
-           .Get()) {
-    return;
-  }
-
-  v8::Local<v8::External> v8_this =
-      v8::External::New(v8_helper_->isolate(), this);
-
-  v8::Local<v8::Object> private_aggregation =
-      v8::Object::New(v8_helper_->isolate());
-
-  v8::Local<v8::Function> send_histogram_report_function =
-      v8::Function::New(
-          context, &PrivateAggregationBindings::ContributeToHistogram, v8_this)
-          .ToLocalChecked();
-  private_aggregation
-      ->Set(context,
-            v8_helper_->CreateStringFromLiteral("contributeToHistogram"),
-            send_histogram_report_function)
-      .Check();
-
-    v8::Local<v8::Function> report_contribution_for_event_function =
-        v8::Function::New(
-            context, &PrivateAggregationBindings::ContributeToHistogramOnEvent,
-            v8_this)
-            .ToLocalChecked();
-    private_aggregation
-        ->Set(
-            context,
-            v8_helper_->CreateStringFromLiteral("contributeToHistogramOnEvent"),
-            report_contribution_for_event_function)
-        .Check();
-
-  v8::Local<v8::Function> enable_debug_mode_function =
-      v8::Function::New(context, &PrivateAggregationBindings::EnableDebugMode,
-                        v8_this)
-          .ToLocalChecked();
-  private_aggregation
-      ->Set(context, v8_helper_->CreateStringFromLiteral("enableDebugMode"),
-            enable_debug_mode_function)
-      .Check();
-
-  context->Global()
-      ->Set(context, v8_helper_->CreateStringFromLiteral("privateAggregation"),
-            private_aggregation)
-      .Check();
 }
 
 void PrivateAggregationBindings::Reset() {
