diff --git a/url/url_util.cc b/url/url_util.cc
index f77ce7f6de..9387795bd3 100644
--- a/url/url_util.cc
+++ b/url/url_util.cc
@@ -85,6 +85,7 @@ struct SchemeRegistry {
       kWssScheme,
       kDataScheme,
       kAboutScheme,
+      kTraceScheme,
   };
 
   // Schemes that normal pages cannot link to or access (i.e., with the same
@@ -99,6 +100,7 @@ struct SchemeRegistry {
       kAboutScheme,
       kJavaScriptScheme,
       kDataScheme,
+      kTraceScheme,
   };
 
   // Schemes that can be sent CORS requests.
