diff --git a/url/url_constants.h b/url/url_constants.h
index 1c69603914..af00dc5842 100644
--- a/url/url_constants.h
+++ b/url/url_constants.h
@@ -50,6 +50,8 @@ inline constexpr char kMaterializedViewScheme[] = "materialized-view";
 inline constexpr char kSteamScheme[] = "steam";
 inline constexpr char kTelScheme[] = "tel";
 inline constexpr char16_t kTelScheme16[] = u"tel";
+inline constexpr char kTraceScheme[] = "trk";
+inline constexpr char16_t kTraceScheme16[] = u"trk";
 inline constexpr char kUrnScheme[] = "urn";
 inline constexpr char16_t kUrnScheme16[] = u"urn";
 inline constexpr char kUuidInPackageScheme[] = "uuid-in-package";
