diff --git a/components/safety_check/safety_check.cc b/components/safety_check/safety_check.cc
index df53469b14..6d2ea57fd6 100644
--- a/components/safety_check/safety_check.cc
+++ b/components/safety_check/safety_check.cc
@@ -4,26 +4,9 @@
 
 #include "components/safety_check/safety_check.h"
 
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
-
 namespace safety_check {
 
 SafeBrowsingStatus CheckSafeBrowsing(PrefService* pref_service) {
-  const PrefService::Preference* enabled_pref =
-      pref_service->FindPreference(prefs::kSafeBrowsingEnabled);
-  bool is_sb_enabled = pref_service->GetBoolean(prefs::kSafeBrowsingEnabled);
-  bool is_sb_managed = enabled_pref->IsManaged();
-
-  if (is_sb_enabled && pref_service->GetBoolean(prefs::kSafeBrowsingEnhanced))
-    return SafeBrowsingStatus::kEnabledEnhanced;
-  if (is_sb_enabled && is_sb_managed)
-    return SafeBrowsingStatus::kEnabledStandard;
-  if (is_sb_enabled && !is_sb_managed)
-    return SafeBrowsingStatus::kEnabledStandardAvailableEnhanced;
-  if (is_sb_managed)
-    return SafeBrowsingStatus::kDisabledByAdmin;
-  if (enabled_pref->IsExtensionControlled())
-    return SafeBrowsingStatus::kDisabledByExtension;
   return SafeBrowsingStatus::kDisabled;
 }
 
