diff --git a/chrome/browser/notifications/persistent_notification_handler.cc b/chrome/browser/notifications/persistent_notification_handler.cc
index cd614acd70..38a92b279b 100644
--- a/chrome/browser/notifications/persistent_notification_handler.cc
+++ b/chrome/browser/notifications/persistent_notification_handler.cc
@@ -314,24 +314,6 @@ void PersistentNotificationHandler::DisableNotifications(
   NotificationPermissionContext::UpdatePermission(profile, origin,
                                                   CONTENT_SETTING_BLOCK);
 #endif
-  // Remove `origin` from user allowlisted sites when user unsubscribes. On
-  // Android, log the suspicious notification unsubscribe ukm event if the
-  // notification was suspicious.
-  auto* hcsm = HostContentSettingsMapFactory::GetForProfile(profile);
-  if (hcsm && origin.is_valid()) {
-    hcsm->SetWebsiteSettingCustomScope(
-        ContentSettingsPattern::FromURLNoWildcard(origin),
-        ContentSettingsPattern::Wildcard(),
-        ContentSettingsType::ARE_SUSPICIOUS_NOTIFICATIONS_ALLOWLISTED_BY_USER,
-        base::Value(base::Value::Dict().Set(
-            safe_browsing::kIsAllowlistedByUserKey, false)));
-#if BUILDFLAG(IS_ANDROID)
-    if (notification_id.has_value()) {
-      safe_browsing::MaybeLogSuspiciousNotificationUnsubscribeUkm(
-          hcsm, origin, notification_id.value(), profile);
-    }
-#endif
-  }
 }
 
 void PersistentNotificationHandler::OpenSettings(Profile* profile,
@@ -384,42 +366,6 @@ void PersistentNotificationHandler::OnMaybeReport(
     Profile* profile,
     bool did_show_warning,
     bool did_user_unsubscribe) {
-  CHECK(profile);
-
-  // In case the data volume becomes excessive, logging should happen at a
-  // sampled rate. This rate is defined by the
-  // `kReportNotificationContentDetectionDataRate` feature parameter.
-  if (base::RandDouble() * 100 >
-      safe_browsing::kReportNotificationContentDetectionDataRate.Get()) {
-    return;
-  }
-
-  scoped_refptr<content::PlatformNotificationContext> notification_context =
-      profile->GetStoragePartitionForUrl(url)->GetPlatformNotificationContext();
-  if (!notification_context ||
-      !OptimizationGuideKeyedServiceFactory::GetForProfile(profile)) {
-    return;
-  }
-
-  blink::mojom::EngagementLevel engagement_level =
-      blink::mojom::EngagementLevel::NONE;
-  if (site_engagement::SiteEngagementService::Get(profile)) {
-    engagement_level = site_engagement::SiteEngagementService::Get(profile)
-                           ->GetEngagementLevel(url);
-  }
-
-  // Read notification data from database and upload as log to model quality
-  // service.
-  notification_context->ReadNotificationDataAndRecordInteraction(
-      notification_id, url,
-      content::PlatformNotificationContext::Interaction::NONE,
-      base::BindOnce(
-          &safe_browsing::SendNotificationContentDetectionDataToMQLSServer,
-          OptimizationGuideKeyedServiceFactory::GetForProfile(profile)
-              ->GetModelQualityLogsUploaderService()
-              ->GetWeakPtr(),
-          safe_browsing::NotificationContentDetectionMQLSMetadata(
-              did_show_warning, did_user_unsubscribe, engagement_level)));
 }
 
 #if BUILDFLAG(ENABLE_BACKGROUND_MODE)
