diff --git a/extensions/browser/api/declarative_net_request/declarative_net_request_api.cc b/extensions/browser/api/declarative_net_request/declarative_net_request_api.cc
index 54268f44e2..774cdc8d40 100644
--- a/extensions/browser/api/declarative_net_request/declarative_net_request_api.cc
+++ b/extensions/browser/api/declarative_net_request/declarative_net_request_api.cc
@@ -135,13 +135,6 @@ DeclarativeNetRequestUpdateDynamicRulesFunction::Run() {
     return RespondNow(NoArguments());
   }
 
-  // Collect rules to add in the Extension Telemetry Service.
-  if (!rules_to_add.empty()) {
-    ExtensionsBrowserClient::Get()
-        ->GetSafeBrowsingDelegate()
-        ->NotifyExtensionApiDeclarativeNetRequest(browser_context(),
-                                                  extension_id(), rules_to_add);
-  }
 
   auto* rules_monitor_service =
       declarative_net_request::RulesMonitorService::Get(browser_context());
@@ -249,13 +242,6 @@ DeclarativeNetRequestUpdateSessionRulesFunction::Run() {
     return RespondNow(NoArguments());
   }
 
-  // Collect rules to add in the Extension Telemetry Service.
-  if (!rules_to_add.empty()) {
-    ExtensionsBrowserClient::Get()
-        ->GetSafeBrowsingDelegate()
-        ->NotifyExtensionApiDeclarativeNetRequest(browser_context(),
-                                                  extension_id(), rules_to_add);
-  }
 
   auto* rules_monitor_service =
       declarative_net_request::RulesMonitorService::Get(browser_context());
