diff --git a/chrome/browser/signin/android/signin_manager_android.cc b/chrome/browser/signin/android/signin_manager_android.cc
index b2527e40c6..e906aa6eb1 100644
--- a/chrome/browser/signin/android/signin_manager_android.cc
+++ b/chrome/browser/signin/android/signin_manager_android.cc
@@ -33,7 +33,6 @@
 #include "components/policy/core/common/policy_switches.h"
 #include "components/prefs/android/pref_service_android.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/account_managed_status_finder.h"
 #include "components/signin/public/identity_manager/accounts_cookie_mutator.h"
@@ -116,11 +115,6 @@ class ProfileDataRemover : public content::BrowsingDataRemover::Observer {
       // All the Profile data has been wiped. Clear the last signed in username
       // as well, so that the next signin doesn't trigger the account
       // change dialog.
-      profile_->GetPrefs()->ClearPref(prefs::kGoogleServicesLastSyncingGaiaId);
-      profile_->GetPrefs()->ClearPref(
-          prefs::kGoogleServicesLastSyncingUsername);
-      profile_->GetPrefs()->ClearPref(
-          prefs::kGoogleServicesLastSignedInUsername);
     }
 
     origin_runner_->PostTask(FROM_HERE, std::move(callback_));
