diff --git a/components/collaboration/internal/collaboration_service_impl.cc b/components/collaboration/internal/collaboration_service_impl.cc
index b7fbf74315..73044cbf72 100644
--- a/components/collaboration/internal/collaboration_service_impl.cc
+++ b/components/collaboration/internal/collaboration_service_impl.cc
@@ -20,7 +20,6 @@
 #include "components/prefs/pref_service.h"
 #include "components/saved_tab_groups/public/tab_group_sync_service.h"
 #include "components/signin/public/base/consent_level.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/sync/base/collaboration_id.h"
 #include "components/sync/base/features.h"
@@ -68,10 +67,6 @@ CollaborationServiceImpl::CollaborationServiceImpl(
       prefs::kSharedTabGroupsManagedAccountSetting,
       base::BindRepeating(&CollaborationServiceImpl::RefreshServiceStatus,
                           base::Unretained(this)));
-  registrar_.Add(
-      ::prefs::kSigninAllowed,
-      base::BindRepeating(&CollaborationServiceImpl::RefreshServiceStatus,
-                          base::Unretained(this)));
 }
 
 CollaborationServiceImpl::~CollaborationServiceImpl() {
@@ -410,144 +405,13 @@ SigninStatus CollaborationServiceImpl::GetSigninStatus() {
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
   SigninStatus status = SigninStatus::kNotSignedIn;
 
-  bool has_valid_primary_account =
-      identity_manager_->HasPrimaryAccountWithRefreshToken(
-          signin::ConsentLevel::kSignin) &&
-      !identity_manager_->HasAccountWithRefreshTokenInPersistentErrorState(
-          identity_manager_->GetPrimaryAccountId(
-              signin::ConsentLevel::kSignin));
-  if (has_valid_primary_account) {
-    status = SigninStatus::kSignedIn;
-  } else if (identity_manager_->HasPrimaryAccount(
-                 signin::ConsentLevel::kSignin)) {
-    status = SigninStatus::kSignedInPaused;
-  } else if (!profile_prefs_->GetBoolean(::prefs::kSigninAllowed)) {
     status = SigninStatus::kSigninDisabled;
-  }
-
-#if BUILDFLAG(IS_IOS)
-  BrowserSigninMode policy_mode = static_cast<BrowserSigninMode>(
-      local_prefs_->GetInteger(::prefs::kBrowserSigninPolicy));
-  if (policy_mode == BrowserSigninMode::kDisabled) {
-    status = SigninStatus::kSigninDisabled;
-  }
-#endif
 
   return status;
 }
 
 CollaborationStatus CollaborationServiceImpl::GetCollaborationStatus() {
-  DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
-  if (!base::FeatureList::IsEnabled(
-          data_sharing::features::kDataSharingFeature) &&
-      !base::FeatureList::IsEnabled(
-          data_sharing::features::kDataSharingJoinOnly)) {
     return CollaborationStatus::kDisabled;
-  }
-
-  // Check if version out-of-date turn off shared tab groups data types and show
-  // update chrome ui.
-  if (base::FeatureList::IsEnabled(
-          data_sharing::features::kSharedDataTypesKillSwitch)) {
-    return base::FeatureList::IsEnabled(
-               data_sharing::features::kDataSharingEnableUpdateChromeUI)
-               ? CollaborationStatus::kVersionOutOfDateShowUpdateChromeUi
-               : CollaborationStatus::kVersionOutOfDate;
-  }
-
-  // Check if device policy allow signin.
-#if BUILDFLAG(IS_IOS)
-  BrowserSigninMode policy_mode = static_cast<BrowserSigninMode>(
-      local_prefs_->GetInteger(::prefs::kBrowserSigninPolicy));
-  if (policy_mode == BrowserSigninMode::kDisabled) {
-    return CollaborationStatus::kDisabledForPolicy;
-  }
-#else
-  if (!profile_prefs_->GetBoolean(::prefs::kSigninAllowed) &&
-      profile_prefs_->IsManagedPreference(::prefs::kSigninAllowed)) {
-    return CollaborationStatus::kDisabledForPolicy;
-  }
-#endif
-
-  // Check if device policy allow sync.
-  if (current_status_.sync_status == SyncStatus::kSyncDisabledByEnterprise) {
-    return CollaborationStatus::kDisabledForPolicy;
-  }
-
-  // TODO(haileywang): Support collaboration status updates.
-  CollaborationStatus status = CollaborationStatus::kDisabled;
-  if (base::FeatureList::IsEnabled(
-          data_sharing::features::kDataSharingFeature)) {
-    status = CollaborationStatus::kEnabledCreateAndJoin;
-  } else if (base::FeatureList::IsEnabled(
-                 data_sharing::features::kDataSharingJoinOnly)) {
-    status = CollaborationStatus::kAllowedToJoin;
-  }
-
-  if (current_status_.signin_status == SigninStatus::kNotSignedIn) {
-    return status;
-  }
-
-  CoreAccountInfo account =
-      identity_manager_->GetPrimaryAccountInfo(signin::ConsentLevel::kSignin);
-  if (!signin::AccountManagedStatusFinder::MayBeEnterpriseUserBasedOnEmail(
-          account.email)) {
-    return status;
-  }
-
-  // Enterprise account handling.
-  if (!account_managed_status_finder_) {
-    account_managed_status_finder_ =
-        std::make_unique<signin::AccountManagedStatusFinder>(
-            identity_manager_, account,
-            base::BindOnce(&CollaborationServiceImpl::RefreshServiceStatus,
-                           weak_ptr_factory_.GetWeakPtr()),
-            base::Seconds(5));
-  }
-
-  // Enterprise V2: Check enterprise policy to allow/disallow collaboration
-  // feature.
-  if (base::FeatureList::IsEnabled(
-          data_sharing::features::kCollaborationEntrepriseV2)) {
-    switch (account_managed_status_finder_->GetOutcome()) {
-      case Outcome::kConsumerGmail:
-      case Outcome::kConsumerWellKnown:
-      case Outcome::kConsumerNotWellKnown:
-        break;
-      default:
-        if (profile_prefs_->GetInteger(
-                collaboration::prefs::kSharedTabGroupsManagedAccountSetting) ==
-            static_cast<int>(
-                prefs::SharedTabGroupsManagedAccountSetting::kDisabled)) {
-          return CollaborationStatus::kDisabledForPolicy;
-        }
-    }
-
-    return status;
-  }
-
-  // Enterprise V1: Figure out if collaboration feature is disabled by account
-  // policy. This early check allows to not disable collaboration feature when
-  // the user need to refresh their account (refresh tokens unavailable).
-  switch (account_managed_status_finder_->GetOutcome()) {
-    case Outcome::kPending:
-      status = CollaborationStatus::kDisabledPending;
-      break;
-    case Outcome::kError:
-    case Outcome::kTimeout:
-      status = CollaborationStatus::kDisabled;
-      break;
-    case Outcome::kEnterpriseGoogleDotCom:
-    case Outcome::kEnterprise:
-      status = CollaborationStatus::kDisabledForPolicy;
-      break;
-    case Outcome::kConsumerGmail:
-    case Outcome::kConsumerWellKnown:
-    case Outcome::kConsumerNotWellKnown:
-      break;
-  }
-
-  return status;
 }
 
 void CollaborationServiceImpl::RefreshServiceStatus() {
