diff --git a/chrome/browser/safe_browsing/advanced_protection_status_manager_desktop.cc b/chrome/browser/safe_browsing/advanced_protection_status_manager_desktop.cc
index 6ca2403759..89aeae918e 100644
--- a/chrome/browser/safe_browsing/advanced_protection_status_manager_desktop.cc
+++ b/chrome/browser/safe_browsing/advanced_protection_status_manager_desktop.cc
@@ -15,7 +15,6 @@
 #include "chrome/browser/signin/identity_manager_factory.h"
 #include "components/prefs/pref_service.h"
 #include "components/safe_browsing/core/common/features.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/common/safebrowsing_switches.h"
 #include "components/signin/public/base/consent_level.h"
 #include "components/signin/public/identity_manager/account_info.h"
@@ -72,9 +71,7 @@ void AdvancedProtectionStatusManagerDesktop::MaybeRefreshOnStartUp() {
   RecordStartupUma(is_under_advanced_protection_);
   NotifyObserversStatusChanged();
 
-  if (pref_service_->HasPrefPath(prefs::kAdvancedProtectionLastRefreshInUs)) {
-    last_refreshed_ = base::Time::FromDeltaSinceWindowsEpoch(base::Microseconds(
-        pref_service_->GetInt64(prefs::kAdvancedProtectionLastRefreshInUs)));
+  if (false) {
     if (is_under_advanced_protection_) {
       ScheduleNextRefresh();
     }
@@ -96,16 +93,7 @@ AdvancedProtectionStatusManagerDesktop::
     ~AdvancedProtectionStatusManagerDesktop() = default;
 
 bool AdvancedProtectionStatusManagerDesktop::IsUnderAdvancedProtection() const {
-  if (!pref_service_->GetBoolean(prefs::kAdvancedProtectionAllowed)) {
     return false;
-  }
-
-  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kForceTreatUserAsAdvancedProtection)) {
-    return true;
-  }
-
-  return is_under_advanced_protection_;
 }
 
 void AdvancedProtectionStatusManagerDesktop::
@@ -261,10 +249,6 @@ void AdvancedProtectionStatusManagerDesktop::CancelFutureRefresh() {
 }
 
 void AdvancedProtectionStatusManagerDesktop::UpdateLastRefreshTime() {
-  last_refreshed_ = base::Time::Now();
-  pref_service_->SetInt64(
-      prefs::kAdvancedProtectionLastRefreshInUs,
-      last_refreshed_.ToDeltaSinceWindowsEpoch().InMicroseconds());
 }
 
 bool AdvancedProtectionStatusManagerDesktop::IsUnconsentedPrimaryAccount(
