diff --git a/chrome/browser/safe_browsing/chrome_password_protection_service.cc b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
index 8061c06531..fcb6eb5421 100644
--- a/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+++ b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
@@ -72,7 +72,6 @@
 #include "components/safe_browsing/core/browser/verdict_cache_manager.h"
 #include "components/safe_browsing/core/common/features.h"
 #include "components/safe_browsing/core/common/proto/csd.pb.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/common/safebrowsing_constants.h"
 #include "components/safe_browsing/core/common/utils.h"
 #include "components/security_interstitials/core/unsafe_resource.h"
@@ -1373,14 +1372,6 @@ bool ChromePasswordProtectionService::HasUnhandledEnterprisePasswordReuse(
 }
 
 void ChromePasswordProtectionService::OnWarningTriggerChanged() {
-  const base::Value& pref_value = pref_change_registrar_->prefs()->GetValue(
-      prefs::kPasswordProtectionWarningTrigger);
-  // If password protection is not turned off, do nothing.
-  if (static_cast<PasswordProtectionTrigger>(pref_value.GetInt()) !=
-      PASSWORD_PROTECTION_OFF) {
-    return;
-  }
-
   // Clears captured enterprise password hashes or GSuite sync password hashes.
   password_manager::PasswordReuseManager* reuse_manager =
       GetPasswordReuseManager();
@@ -1553,7 +1544,7 @@ PrefService* ChromePasswordProtectionService::GetPrefs() const {
 }
 
 bool ChromePasswordProtectionService::IsSafeBrowsingEnabled() {
-  return ::safe_browsing::IsSafeBrowsingEnabled(*GetPrefs());
+  return false;
 }
 
 bool ChromePasswordProtectionService::IsExtendedReporting() {
@@ -1573,39 +1564,7 @@ bool ChromePasswordProtectionService::IsInPasswordAlertMode(
 bool ChromePasswordProtectionService::IsPingingEnabled(
     LoginReputationClientRequest::TriggerType trigger_type,
     ReusedPasswordAccountType password_type) {
-  if (!IsSafeBrowsingEnabled()) {
-    return false;
-  }
-  bool extended_reporting_enabled = IsExtendedReporting();
-  if (trigger_type == LoginReputationClientRequest::PASSWORD_REUSE_EVENT) {
-    // Don't send a ping if the password protection setting is off
-    if (GetPasswordProtectionWarningTriggerPref(password_type) ==
-        PASSWORD_PROTECTION_OFF) {
-      return false;
-    }
-    // If the account type is UNKNOWN (i.e. AccountInfo fields could not be
-    // retrieved from server), pings should be gated by SBER.
-    if (password_type.account_type() == ReusedPasswordAccountType::UNKNOWN) {
-      return extended_reporting_enabled;
-    }
-
-// Only saved password and GAIA password reuse warnings are shown to users on
-// Android, so other types of password reuse events should be gated by Safe
-// Browsing extended reporting.
-#if BUILDFLAG(IS_ANDROID)
-    if (password_type.account_type() ==
-            ReusedPasswordAccountType::SAVED_PASSWORD ||
-        password_type.account_type() == ReusedPasswordAccountType::GMAIL) {
-      return true;
-    }
-
-    return extended_reporting_enabled;
-#else
-    return true;
-#endif
-  }
-
-  return !IsIncognito() && extended_reporting_enabled;
+  return false;
 }
 
 RequestOutcome ChromePasswordProtectionService::GetPingNotSentReason(
@@ -1847,23 +1806,6 @@ MaybeCreateCommitDeferringCondition(
              : nullptr;
 }
 
-PasswordProtectionTrigger
-ChromePasswordProtectionService::GetPasswordProtectionWarningTriggerPref(
-    ReusedPasswordAccountType password_type) const {
-  bool is_policy_managed = profile_->GetPrefs()->HasPrefPath(
-      prefs::kPasswordProtectionWarningTrigger);
-  PasswordProtectionTrigger trigger_level =
-      static_cast<PasswordProtectionTrigger>(profile_->GetPrefs()->GetInteger(
-          prefs::kPasswordProtectionWarningTrigger));
-  if (is_policy_managed && trigger_level == PASSWORD_PROTECTION_OFF) {
-    return PASSWORD_PROTECTION_OFF;
-  }
-  if (password_type.account_type() == ReusedPasswordAccountType::GMAIL) {
-    return PHISHING_REUSE;
-  }
-  return is_policy_managed ? trigger_level : PHISHING_REUSE;
-}
-
 bool ChromePasswordProtectionService::IsURLAllowlistedForPasswordEntry(
     const GURL& url) const {
   if (!profile_)
