diff --git a/chrome/browser/webdata_services/web_data_service_factory.cc b/chrome/browser/webdata_services/web_data_service_factory.cc
index 3993796992..55933f0a26 100644
--- a/chrome/browser/webdata_services/web_data_service_factory.cc
+++ b/chrome/browser/webdata_services/web_data_service_factory.cc
@@ -18,7 +18,6 @@
 #include "components/plus_addresses/webdata/plus_address_webdata_service.h"
 #include "components/prefs/pref_service.h"
 #include "components/search_engines/keyword_web_data_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/webdata/token_web_data.h"
 #include "components/webdata_services/web_data_service_wrapper.h"
 #include "content/public/browser/browser_task_traits.h"
@@ -89,51 +88,7 @@ enum class AutofillAccountStorageResult {
 AutofillAccountStorageResult DetermineAutofillAccountStorage(
     PrefService* pref_service) {
   CHECK(pref_service);
-  // The interpretation of the pref mimics what PrimaryAccountManager's
-  // constructor does.
-  const bool is_signed_in =
-      !pref_service->GetString(::prefs::kGoogleServicesAccountId).empty();
-  // If the user is signed out during profile startup, as per switch above
-  // being enabled, any new sign-ins will involve an explicit sign-in (i.e.
-  // interaction with native UI). In this case, on-disk storage is appropriate.
-  if (!is_signed_in) {
     return AutofillAccountStorageResult::kOnDisk_SignedOut;
-  }
-  // It is possible that the user already is in an explicit sign-in state. In
-  // this case, on-disk storage is appropriate, as any additional future
-  // sign-ins (if the user first signs out) are guaranteed to be explicit
-  // sign-ins too.
-  if (pref_service->GetBoolean(::prefs::kExplicitBrowserSignin)) {
-    return AutofillAccountStorageResult::kOnDisk_SignedInExplicitly;
-  }
-  // The interpretation of the pref mimics what PrimaryAccountManager's
-  // constructor does.
-  const bool is_consented_to_sync =
-      pref_service->GetBoolean(::prefs::kGoogleServicesConsentedToSync);
-  // If Sync (the feature) is on, the account storage isn't currently used. This
-  // is because the only way to activate the account storate requires signing
-  // out first, which means the predicate can return false as per earlier
-  // rationale. With one exception: managed profiles may turn sync off without
-  // signing out. Either way, having turned sync on implies the user interacted
-  // explicitly with a sync UI, so in this particular context it is no different
-  // from explicit sign-in, and on-disk storage is appropriate.
-  if (is_consented_to_sync) {
-    return AutofillAccountStorageResult::kOnDisk_SyncFeatureEnabled;
-  }
-  // The remaining case implies a legacy signed-in-non-syncing state with
-  // implicit sign-in, which means the user signed in before the latest feature
-  // flags rolled out. This is the only case where in-memory storage should be
-  // used.
-  //
-  // Note that, during the lifetime of the browser/profile, it is still possible
-  // that the users signs out and signs back in, where the latter is guaranteed
-  // to be an explicit sign-in. In this case, it would be theoretically better
-  // to immediately switch to on-disk storage, but this isn't possible once a
-  // profile is initialized (as this predicate only gets evaluated once).
-  // Conveniently, it is also harmless to use the in-memory storage until the
-  // next browser restart, given that this is a one-off transition (upon restart
-  // the code would run into one of the cases listed earlier that return false).
-  return AutofillAccountStorageResult::kInMemory_SignedInImplicitly;
 }
 #endif  // !BUILDFLAG(IS_ANDROID)
 
