diff --git a/extensions/browser/api/execute_code_function.cc b/extensions/browser/api/execute_code_function.cc
index abe1d962c5..274e213074 100644
--- a/extensions/browser/api/execute_code_function.cc
+++ b/extensions/browser/api/execute_code_function.cc
@@ -177,12 +177,6 @@ ExtensionFunction::ResponseAction ExecuteCodeFunction::Run() {
     return RespondNow(Error(std::move(error)));
 
   if (details_->code) {
-    if (!IsWebView() && extension()) {
-      ExtensionsBrowserClient::Get()
-          ->GetSafeBrowsingDelegate()
-          ->NotifyExtensionApiTabExecuteScript(browser_context(),
-                                               extension_id(), *details_->code);
-    }
 
     if (!Execute(*details_->code, &error))
       return RespondNow(Error(std::move(error)));
