diff --git a/net/dns/public/doh_provider_entry.cc b/net/dns/public/doh_provider_entry.cc
index f79af28e8d..163339b14e 100644
--- a/net/dns/public/doh_provider_entry.cc
+++ b/net/dns/public/doh_provider_entry.cc
@@ -171,30 +171,6 @@ const DohProviderEntry::List& DohProviderEntry::GetList() {
         /*display_globally=*/false,
         /*display_countries=*/{"EE", "DE"},
         LoggingLevel::kNormal},
-       {"Google",
-        MAKE_BASE_FEATURE_WITH_STATIC_STORAGE(DohProviderGoogle,
-                                              base::FEATURE_ENABLED_BY_DEFAULT),
-        {"8.8.8.8", "8.8.4.4", "2001:4860:4860::8888", "2001:4860:4860::8844"},
-        /*dns_over_tls_hostnames=*/
-        {"dns.google", "dns.google.com", "8888.google"},
-        "https://dns.google/dns-query{?dns}",
-        /*ui_name=*/"Google (Public DNS)",
-        "https://developers.google.com/speed/public-dns/"
-        /*privacy_policy=*/"privacy",
-        /*display_globally=*/true,
-        /*display_countries=*/{},
-        LoggingLevel::kExtra},
-       {"GoogleDns64",
-        MAKE_BASE_FEATURE_WITH_STATIC_STORAGE(DohProviderGoogleDns64,
-                                              base::FEATURE_ENABLED_BY_DEFAULT),
-        {"2001:4860:4860::64", "2001:4860:4860::6464"},
-        /*dns_over_tls_hostnames=*/{"dns64.dns.google"},
-        "https://dns64.dns.google/dns-query{?dns}",
-        /*ui_name=*/"",
-        /*privacy_policy=*/"",
-        /*display_globally=*/false,
-        /*display_countries=*/{},
-        LoggingLevel::kNormal},
        {"Iij",
         MAKE_BASE_FEATURE_WITH_STATIC_STORAGE(DohProviderIij,
                                               base::FEATURE_ENABLED_BY_DEFAULT),
