diff --git a/chrome/browser/profiles/BUILD.gn b/chrome/browser/profiles/BUILD.gn
index 8354b98076..f6ba4f491f 100644
--- a/chrome/browser/profiles/BUILD.gn
+++ b/chrome/browser/profiles/BUILD.gn
@@ -324,7 +324,6 @@ source_set("profiles_extra_parts_impl") {
     ]
   } else {
     deps += [
-      "//chrome/browser/accessibility/tree_fixing:service",
       "//chrome/browser/apps/app_service",
       "//chrome/browser/autofill",
       "//chrome/browser/contextual_cueing",
@@ -332,7 +331,6 @@ source_set("profiles_extra_parts_impl") {
       "//chrome/browser/feedback",
       "//chrome/browser/hid",
       "//chrome/browser/media/router/discovery/access_code:access_code_sink_service",
-      "//chrome/browser/screen_ai:screen_ai_service_router_factory",
       "//chrome/browser/search",
       "//chrome/browser/search/background",
       "//chrome/browser/search_engine_choice",
