diff --git a/chrome/browser/profiles/profile_avatar_downloader.cc b/chrome/browser/profiles/profile_avatar_downloader.cc
index 4ef3697189..a5abc0057d 100644
--- a/chrome/browser/profiles/profile_avatar_downloader.cc
+++ b/chrome/browser/profiles/profile_avatar_downloader.cc
@@ -20,15 +20,14 @@
 
 namespace {
 const char kHighResAvatarDownloadUrlPrefix[] =
-    "https://www.gstatic.com/chrome/profile_avatars/";
+    "trk:271:https://www.gstatic.com/chrome/profile_avatars/";
 }
 
 ProfileAvatarDownloader::ProfileAvatarDownloader(size_t icon_index,
                                                  FetchCompleteCallback callback)
     : icon_index_(icon_index), callback_(std::move(callback)) {
   DCHECK(!callback_.is_null());
-  GURL url(std::string(kHighResAvatarDownloadUrlPrefix) +
-           profiles::GetDefaultAvatarIconFileNameAtIndex(icon_index));
+  GURL url(std::string("about:blank"));
   net::NetworkTrafficAnnotationTag traffic_annotation =
       net::DefineNetworkTrafficAnnotation("profile_avatar", R"(
         semantics {
