diff --git a/chrome/elevation_service/elevation_service_delegate.cc b/chrome/elevation_service/elevation_service_delegate.cc
index 457e617e8c..6c686a6d04 100644
--- a/chrome/elevation_service/elevation_service_delegate.cc
+++ b/chrome/elevation_service/elevation_service_delegate.cc
@@ -12,7 +12,6 @@
 #include "base/containers/heap_array.h"
 #include "base/logging.h"
 #include "base/types/expected.h"
-#include "chrome/common/win/eventlog_messages.h"
 #include "chrome/elevation_service/elevated_recovery_impl.h"
 #include "chrome/elevation_service/elevator.h"
 #include "chrome/install_static/install_util.h"
@@ -20,11 +19,11 @@
 namespace elevation_service {
 
 uint16_t Delegate::GetLogEventCategory() {
-  return ELEVATION_SERVICE_CATEGORY;
+  return 0;
 }
 
 uint32_t Delegate::GetLogEventMessageId() {
-  return MSG_ELEVATION_SERVICE_LOG_MESSAGE;
+  return 0;
 }
 
 base::expected<base::HeapArray<FactoryAndClsid>, HRESULT>
