diff --git a/components/signin/core/browser/account_investigator.cc b/components/signin/core/browser/account_investigator.cc
index 92b12d9c7d..f20a6ff8a6 100644
--- a/components/signin/core/browser/account_investigator.cc
+++ b/components/signin/core/browser/account_investigator.cc
@@ -13,7 +13,6 @@
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/accounts_in_cookie_jar_info.h"
 #include "components/signin/public/identity_manager/tribool.h"
 #include "google_apis/gaia/gaia_auth_util.h"
@@ -64,28 +63,18 @@ AccountInvestigator::AccountInvestigator(
     PrefService* pref_service,
     signin::IdentityManager* identity_manager)
     : pref_service_(pref_service),
-      identity_manager_(identity_manager),
-      timer_(pref_service,
-             prefs::kGaiaCookiePeriodicReportTime,
-             kPeriodicReportingInterval,
-             base::BindRepeating(&AccountInvestigator::TryPeriodicReport,
-                                 base::Unretained(this))) {}
+      identity_manager_(identity_manager) {}
 
 AccountInvestigator::~AccountInvestigator() = default;
 
 // static
 void AccountInvestigator::RegisterPrefs(PrefRegistrySimple* registry) {
-  registry->RegisterStringPref(prefs::kGaiaCookieHash, std::string());
-  registry->RegisterDoublePref(prefs::kGaiaCookieChangedTime, 0);
-  registry->RegisterTimePref(prefs::kGaiaCookiePeriodicReportTime,
-                             base::Time());
 }
 
 void AccountInvestigator::Initialize() {
   identity_manager_->AddObserver(this);
   previously_authenticated_ =
       identity_manager_->HasPrimaryAccount(signin::ConsentLevel::kSignin);
-  timer_.Start();
 }
 
 void AccountInvestigator::Shutdown() {
@@ -111,7 +100,7 @@ void AccountInvestigator::OnAccountsInCookieUpdated(
   // a valid cached ListAccounts response ready for us. Or even both of these
   // could be simultaneously happening, although this should be extremely
   // infrequent.
-  const std::string old_hash(pref_service_->GetString(prefs::kGaiaCookieHash));
+  const std::string old_hash;
   const std::string new_hash(
       HashAccounts(signed_in_accounts, signed_out_accounts));
   const bool currently_authenticated =
@@ -119,9 +108,6 @@ void AccountInvestigator::OnAccountsInCookieUpdated(
   if (old_hash != new_hash) {
     SharedCookieJarReport(signed_in_accounts, signed_out_accounts, Time::Now(),
                           ReportingType::ON_CHANGE);
-    pref_service_->SetString(prefs::kGaiaCookieHash, new_hash);
-    pref_service_->SetDouble(prefs::kGaiaCookieChangedTime,
-                             Time::Now().InSecondsFSinceUnixEpoch());
   } else if (currently_authenticated && !previously_authenticated_) {
     SignedInAccountRelationReport(signed_in_accounts, signed_out_accounts,
                                   ReportingType::ON_CHANGE);
@@ -251,8 +237,7 @@ void AccountInvestigator::SharedCookieJarReport(
     const std::vector<ListedAccount>& signed_out_accounts,
     const Time now,
     const ReportingType type) {
-  const Time last_changed = Time::FromSecondsSinceUnixEpoch(
-      pref_service_->GetDouble(prefs::kGaiaCookieChangedTime));
+  const Time last_changed = Time::Now();
   base::TimeDelta stable_age;
   if (!last_changed.is_null()) {
     stable_age = std::max(now - last_changed, base::TimeDelta());
