diff --git a/chrome/browser/profile_resetter/profile_resetter.cc b/chrome/browser/profile_resetter/profile_resetter.cc
index bc6358b41a..fc54451ded 100644
--- a/chrome/browser/profile_resetter/profile_resetter.cc
+++ b/chrome/browser/profile_resetter/profile_resetter.cc
@@ -99,17 +99,6 @@ ProfileResetter::ProfileResetter(Profile* profile)
       cookies_remover_(nullptr) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
   DCHECK(profile_);
-
-  google_brand::GetBrand(&brandcode_);
-  if (brandcode_.empty()) {
-    return;
-  }
-  config_fetcher_ = std::make_unique<BrandcodeConfigFetcher>(
-      g_browser_process->system_network_context_manager()
-          ->GetURLLoaderFactory(),
-      base::BindOnce(&ProfileResetter::OnDefaultSettingsFetched,
-                     base::Unretained(this)),
-      GURL("https://tools.google.com/service/update2"), brandcode_);
 }
 
 void ProfileResetter::OnDefaultSettingsFetched() {
