diff --git a/components/signin/internal/identity_manager/primary_account_manager.cc b/components/signin/internal/identity_manager/primary_account_manager.cc
index 1c62d5a631..edb8351984 100644
--- a/components/signin/internal/identity_manager/primary_account_manager.cc
+++ b/components/signin/internal/identity_manager/primary_account_manager.cc
@@ -25,7 +25,6 @@
 #include "components/signin/public/base/account_consistency_method.h"
 #include "components/signin/public/base/signin_client.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_prefs.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/account_info.h"
@@ -208,176 +207,25 @@ PrimaryAccountManager::PrimaryAccountManager(
   DCHECK(account_tracker_service_);
   ScopedPrefCommit scoped_pref_commit(client_->GetPrefs(),
                                       /*commit_on_destroy=*/false);
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-  signin_allowed_.Init(
-      prefs::kSigninAllowed, client_->GetPrefs(),
-      base::BindRepeating(&PrimaryAccountManager::OnSigninAllowedPrefChanged,
-                          base::Unretained(this)));
-#else
-  scoped_pref_commit.ClearPref(prefs::kExplicitBrowserSignin);
-#endif
-
-  // Prepare prefs before loading them.
-  PrepareToLoadPrefs();
-
-  PrefService* prefs = client_->GetPrefs();
-  std::string pref_account_id =
-      prefs->GetString(prefs::kGoogleServicesAccountId);
-  bool pref_consented_to_sync =
-      prefs->GetBoolean(prefs::kGoogleServicesConsentedToSync);
-  LogPrimaryAccountPrefsOnInitialize(pref_account_id, pref_consented_to_sync);
-
-  if (pref_account_id.empty()) {
     SetPrimaryAccountInternal(CoreAccountInfo(), /*consented_to_sync=*/false,
                               scoped_pref_commit);
-  } else {
-    auto [account_info, account_info_state] =
-        GetOrRestorePrimaryAccountInfoOnInitialize(pref_account_id,
-                                                   pref_consented_to_sync);
-    base::UmaHistogramEnumeration(
-        "Signin.PAMInitialize.PrimaryAccountInfoState", account_info_state);
-
-    if (pref_consented_to_sync && !account_info.IsEmpty()) {
-      SetPrimaryAccountInternal(account_info, /*consented_to_sync=*/true,
-                                scoped_pref_commit);
-
-      // Ensure that the last syncing account data is consistent with the
-      // primary account. The last signed-in account data is written inside
-      // SetPrimaryAccountInternal().
-      scoped_pref_commit.SetString(prefs::kGoogleServicesLastSyncingGaiaId,
-                                   account_info.gaia.ToString());
-      scoped_pref_commit.SetString(prefs::kGoogleServicesLastSyncingUsername,
-                                   account_info.email);
-    } else if (ShouldSigninAllowedPrefAffectPrimaryAccount(
-                   pref_consented_to_sync)) {
-      SetPrimaryAccountInternal(CoreAccountInfo(), /*consented_to_sync=*/false,
-                                scoped_pref_commit);
-    } else {
-      SetPrimaryAccountInternal(account_info, /*consented_to_sync=*/false,
-                                scoped_pref_commit);
-    }
-  }
 
   // PrimaryAccountManager is initialized once the primary account and consent
   // level are loaded.
   CHECK(primary_account_.has_value());
-
-  bool migrated_sync_user_to_explicit_sign_in = false;
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-  if (!prefs->GetBoolean(prefs::kExplicitBrowserSignin) &&
-      HasPrimaryAccount(signin::ConsentLevel::kSync)) {
-    // A profile that is opted in to sync can be migrated to explicit browser
-    // sign-in as the user has explicitly signed in to the browser when they
-    // opted in to sync.
-    scoped_pref_commit.SetBoolean(prefs::kExplicitBrowserSignin, true);
-    migrated_sync_user_to_explicit_sign_in = true;
-  }
-#endif
-
-  base::UmaHistogramBoolean("Signin.ExplicitSigninMigration.FromSync",
-                            migrated_sync_user_to_explicit_sign_in);
-
-  // `prefs::kPrefsThemesSearchEnginesAccountStorageEnabled` is set for sync
-  // users and new signed in users. It is not cleared on sign out.
-  if (base::FeatureList::IsEnabled(
-          switches::kEnablePreferencesAccountStorage)) {
-    if (HasPrimaryAccount(signin::ConsentLevel::kSync)) {
-      scoped_pref_commit.SetBoolean(
-          prefs::kPrefsThemesSearchEnginesAccountStorageEnabled, true);
-    }
-  } else {
-    scoped_pref_commit.ClearPref(
-        prefs::kPrefsThemesSearchEnginesAccountStorageEnabled);
-  }
-
-  std::vector<AccountInfo> accounts_in_tracker_service =
-      account_tracker_service_->GetAccounts();
-  SigninPrefs signin_prefs(*prefs);
-
-  for (const auto& account : accounts_in_tracker_service) {
-    // Clear the extensions explicit sign in pref if the feature flag is not
-    // enabled.
-    if (!switches::IsExtensionsExplicitBrowserSigninEnabled()) {
-      signin_prefs.SetExtensionsExplicitBrowserSignin(account.gaia, false);
-    }
-    // Clear the bookmarks explicit sign in pref if the feature flag is not
-    // enabled.
-    if (!base::FeatureList::IsEnabled(
-            switches::kSyncEnableBookmarksInTransportMode)) {
-      signin_prefs.SetBookmarksExplicitBrowserSignin(account.gaia, false);
-    }
-  }
-
-  // It is important to only load credentials after starting to observe the
-  // token service.
-  token_service_observation_.Observe(token_service_);
-  token_service_->LoadCredentials(
-      GetPrimaryAccountId(signin::ConsentLevel::kSignin));
 }
 
 PrimaryAccountManager::~PrimaryAccountManager() = default;
 
 // static
 void PrimaryAccountManager::RegisterProfilePrefs(PrefRegistrySimple* registry) {
-  registry->RegisterStringPref(prefs::kGoogleServicesLastSyncingGaiaId,
-                               std::string());
-  registry->RegisterStringPref(prefs::kGoogleServicesLastSyncingUsername,
-                               std::string());
-  registry->RegisterStringPref(prefs::kGoogleServicesLastSignedInUsername,
-                               std::string());
-  registry->RegisterStringPref(prefs::kGoogleServicesAccountId, std::string());
-  registry->RegisterBooleanPref(prefs::kGoogleServicesConsentedToSync, false);
-  registry->RegisterStringPref(
-      prefs::kGoogleServicesSyncingGaiaIdMigratedToSignedIn, std::string());
-  registry->RegisterStringPref(
-      prefs::kGoogleServicesSyncingUsernameMigratedToSignedIn, std::string());
-  registry->RegisterBooleanPref(prefs::kSigninAllowed, true);
-  registry->RegisterBooleanPref(prefs::kSignedInWithCredentialProvider, false);
-  registry->RegisterBooleanPref(kExplicitBrowserSigninWithoutFeatureEnabled,
-                                false);
-  registry->RegisterBooleanPref(prefs::kExplicitBrowserSignin, false);
-  registry->RegisterBooleanPref(
-      prefs::kPrefsThemesSearchEnginesAccountStorageEnabled, false);
 }
 
 // static
 void PrimaryAccountManager::RegisterPrefs(PrefRegistrySimple* registry) {
-  registry->RegisterStringPref(prefs::kGoogleServicesUsernamePattern,
-                               std::string());
 }
 
 void PrimaryAccountManager::PrepareToLoadPrefs() {
-  // Check this method is only called before loading the primary account.
-  CHECK(!primary_account_.has_value());
-
-  PrefService* prefs = client_->GetPrefs();
-
-  // If the user is clearing the token service from the command line, then
-  // clear their login info also (not valid to be logged in without any
-  // tokens).
-  base::CommandLine* cmd_line = base::CommandLine::ForCurrentProcess();
-  if (cmd_line->HasSwitch(switches::kClearTokenService)) {
-    prefs->SetString(prefs::kGoogleServicesAccountId, "");
-    prefs->SetBoolean(prefs::kGoogleServicesConsentedToSync, false);
-  }
-
-#if BUILDFLAG(IS_CHROMEOS)
-  // Migrate primary account ID from email to Gaia ID if needed.
-  std::string pref_account_id =
-      prefs->GetString(prefs::kGoogleServicesAccountId);
-  if (!pref_account_id.empty()) {
-    if (account_tracker_service_->GetMigrationState() ==
-        AccountTrackerService::MIGRATION_IN_PROGRESS) {
-      CoreAccountInfo account_info =
-          account_tracker_service_->FindAccountInfoByEmail(pref_account_id);
-      // |account_info.gaia| could be empty if |account_id| is already gaia id.
-      if (!account_info.gaia.empty()) {
-        pref_account_id = account_info.gaia.ToString();
-        prefs->SetString(prefs::kGoogleServicesAccountId, pref_account_id);
-      }
-    }
-  }
-#endif
 }
 
 std::pair<CoreAccountInfo, PrimaryAccountManager::InitializeAccountInfoState>
@@ -405,46 +253,10 @@ PrimaryAccountManager::GetOrRestorePrimaryAccountInfoOnInitialize(
                               kEmptyAccountInfo_RestoreFailedNotSyncConsented);
   }
 
-  PrefService* prefs = client_->GetPrefs();
-  const GaiaId last_syncing_gaia_id =
-      GaiaId(prefs->GetString(prefs::kGoogleServicesLastSyncingGaiaId));
-  if (last_syncing_gaia_id.empty()) {
-    return std::make_pair(CoreAccountInfo(),
-                          InitializeAccountInfoState::
-                              kEmptyAccountInfo_RestoreFailedNoLastSyncGaiaId);
-  }
-  std::string last_syncing_email =
-      prefs->GetString(prefs::kGoogleServicesLastSyncingUsername);
-  if (last_syncing_email.empty()) {
-    return std::make_pair(CoreAccountInfo(),
-                          InitializeAccountInfoState::
-                              kEmptyAccountInfo_RestoreFailedNoLastSyncEmail);
-  }
-
-  if (account_id != account_tracker_service_->PickAccountIdForAccount(
-                        last_syncing_gaia_id, last_syncing_email)) {
-    return std::make_pair(
-        CoreAccountInfo(),
-        InitializeAccountInfoState::
-            kEmptyAccountInfo_RestoreFailedAccountIdDontMatch);
-  }
-
-  if (base::FeatureList::IsEnabled(kRestorePrimaryAccountInfo)) {
-    CHECK_EQ(
-        account_id,
-        account_tracker_service_->SeedAccountInfo(
-            last_syncing_gaia_id, last_syncing_email,
-            signin_metrics::AccessPoint::kRestorePrimaryAccountOnProfileLoad));
-
-    return std::make_pair(account_tracker_service_->GetAccountInfo(account_id),
-                          InitializeAccountInfoState::
-                              kEmptyAccountInfo_RestoreSuccessFromLastSyncInfo);
-  } else {
     return std::make_pair(
         CoreAccountInfo(),
         InitializeAccountInfoState::
             kEmptyAccountInfo_RestoreFailedAsRestoreFeatureIsDisabled);
-  }
 }
 
 const PrimaryAccountManager::PrimaryAccount&
@@ -535,10 +347,9 @@ void PrimaryAccountManager::SetSyncPrimaryAccountInternal(
 
 #if DCHECK_IS_ON()
   {
-    std::string pref_account_id =
-        client_->GetPrefs()->GetString(prefs::kGoogleServicesAccountId);
+    std::string pref_account_id;
     bool consented_to_sync =
-        client_->GetPrefs()->GetBoolean(prefs::kGoogleServicesConsentedToSync);
+        false;
 
     DCHECK(pref_account_id.empty() || !consented_to_sync ||
            pref_account_id == account_info.account_id.ToString())
@@ -549,14 +360,6 @@ void PrimaryAccountManager::SetSyncPrimaryAccountInternal(
 
   SetPrimaryAccountInternal(account_info, /*consented_to_sync=*/true,
                             scoped_pref_commit);
-
-  // Go ahead and update the last signed in account info here as well. Once a
-  // user is signed in the corresponding preferences should match. Doing it here
-  // as opposed to on signin allows us to catch the upgrade scenario.
-  scoped_pref_commit.SetString(prefs::kGoogleServicesLastSyncingGaiaId,
-                               account_info.gaia.ToString());
-  scoped_pref_commit.SetString(prefs::kGoogleServicesLastSyncingUsername,
-                               account_info.email);
 }
 
 void PrimaryAccountManager::SetPrimaryAccountInternal(
@@ -568,22 +371,6 @@ void PrimaryAccountManager::SetPrimaryAccountInternal(
   // 'account_info' might be a reference to the contents of `primary_account_`.
   // Create a PrimaryAccount object before calling emplace to avoid crashes.
   primary_account_.emplace(PrimaryAccount(account_info, consented_to_sync));
-  std::string account_id =
-      GetPrimaryAccount().account_info.account_id.ToString();
-  scoped_pref_commit.SetString(prefs::kGoogleServicesAccountId, account_id);
-  scoped_pref_commit.SetBoolean(prefs::kGoogleServicesConsentedToSync,
-                                GetPrimaryAccount().consented_to_sync);
-  // If this was a sign-out (account ID is empty), also clear the "account was
-  // migrated" prefs.
-  if (account_id.empty()) {
-    scoped_pref_commit.ClearPref(
-        prefs::kGoogleServicesSyncingGaiaIdMigratedToSignedIn);
-    scoped_pref_commit.ClearPref(
-        prefs::kGoogleServicesSyncingUsernameMigratedToSignedIn);
-  } else {
-    scoped_pref_commit.SetString(prefs::kGoogleServicesLastSignedInUsername,
-                                 account_info.email);
-  }
 }
 
 void PrimaryAccountManager::UpdatePrimaryAccountInfo() {
@@ -721,108 +508,6 @@ PrimaryAccountChangeEvent::State PrimaryAccountManager::GetPrimaryAccountState()
 void PrimaryAccountManager::ComputeExplicitBrowserSignin(
     const PrimaryAccountChangeEvent& event_details,
     ScopedPrefCommit& scoped_pref_commit) {
-  switch (event_details.GetEventTypeFor(signin::ConsentLevel::kSignin)) {
-    case PrimaryAccountChangeEvent::Type::kNone:
-      break;
-    case PrimaryAccountChangeEvent::Type::kCleared:
-      scoped_pref_commit.ClearPref(kExplicitBrowserSigninWithoutFeatureEnabled);
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-      scoped_pref_commit.ClearPref(prefs::kExplicitBrowserSignin);
-#endif
-      return;
-    case PrimaryAccountChangeEvent::Type::kSet:
-      CHECK(event_details.GetSetPrimaryAccountAccessPoint().has_value());
-      signin_metrics::AccessPoint access_point =
-          event_details.GetSetPrimaryAccountAccessPoint().value();
-      GaiaId current_gaia_id =
-          event_details.GetCurrentState().primary_account.gaia;
-
-      bool is_implicit_signin =
-          access_point == signin_metrics::AccessPoint::kUnknown ||
-          access_point == signin_metrics::AccessPoint::kWebSignin;
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-      if (base::FeatureList::IsEnabled(
-              switches::kWebSigninLeadsToImplicitlySignedInState)) {
-        // To allow easier testing, consider the following access points as
-        // implicit sign-in.
-        is_implicit_signin =
-            is_implicit_signin ||
-            access_point ==
-                signin_metrics::AccessPoint::kChromeSigninInterceptBubble ||
-            access_point ==
-                signin_metrics::AccessPoint::kSigninChoiceRemembered;
-      }
-#endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
-      if (is_implicit_signin) {
-        scoped_pref_commit.ClearPref(
-            kExplicitBrowserSigninWithoutFeatureEnabled);
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-        scoped_pref_commit.ClearPref(prefs::kExplicitBrowserSignin);
-#endif
-        // Reset explicit sign-in prefs for the relevant data types.
-        scoped_pref_commit.SetBoolean(
-            prefs::kPrefsThemesSearchEnginesAccountStorageEnabled, false);
-        SigninPrefs(*client_->GetPrefs())
-            .SetExtensionsExplicitBrowserSignin(current_gaia_id, false);
-        SigninPrefs(*client_->GetPrefs())
-            .SetBookmarksExplicitBrowserSignin(current_gaia_id, false);
-        break;
-      }
-      // All others access points are explicit sign ins except the Web
-      // Signin event.
-      scoped_pref_commit.SetBoolean(kExplicitBrowserSigninWithoutFeatureEnabled,
-                                    true);
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-      scoped_pref_commit.SetBoolean(prefs::kExplicitBrowserSignin, true);
-#endif
-      if (base::FeatureList::IsEnabled(
-              switches::kEnablePreferencesAccountStorage)) {
-        scoped_pref_commit.SetBoolean(
-            prefs::kPrefsThemesSearchEnginesAccountStorageEnabled, true);
-      }
-      if (access_point ==
-              signin_metrics::AccessPoint::kExtensionInstallBubble &&
-          switches::IsExtensionsExplicitBrowserSigninEnabled()) {
-        // Record an opt in for the extensions explicit signin feature and use
-        // the existing pref to determine if it's a new or existing opt in.
-        bool is_new_opt_in =
-            !SigninPrefs(*client_->GetPrefs())
-                 .GetExtensionsExplicitBrowserSignin(current_gaia_id);
-        base::UmaHistogramBoolean(
-            "Signin.Extensions.ExplicitSigninFromExtensionInstallBubble",
-            is_new_opt_in);
-
-        SigninPrefs(*client_->GetPrefs())
-            .SetExtensionsExplicitBrowserSignin(current_gaia_id, true);
-      }
-      if (access_point == signin_metrics::AccessPoint::kBookmarkBubble &&
-          base::FeatureList::IsEnabled(
-              switches::kSyncEnableBookmarksInTransportMode)) {
-        // Record an explicit signin for bookmarks for this account only.
-        SigninPrefs(*client_->GetPrefs())
-            .SetBookmarksExplicitBrowserSignin(current_gaia_id, true);
-      }
-  }
-
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-  // If the user turns on sync, disable account storage for bookmarks. This
-  // way the user does not get duplicate data if they turn off sync (and
-  // choose to preserve their data locally) and then sign in again.
-  // This is safe to remove with the deprecation of
-  // `signin::ConsentLevel::kSync`.
-  if (event_details.GetEventTypeFor(signin::ConsentLevel::kSync) ==
-      signin::PrimaryAccountChangeEvent::Type::kSet) {
-    auto current_gaia_id = event_details.GetCurrentState().primary_account.gaia;
-    auto prefs = SigninPrefs(*client_->GetPrefs());
-
-    if (prefs.GetBookmarksExplicitBrowserSignin(current_gaia_id)) {
-      base::UmaHistogramBoolean(
-          "Signin.Bookmarks.SyncTurnedOnWithAccountStorageEnabled", true);
-    }
-
-    prefs.SetBookmarksExplicitBrowserSignin(current_gaia_id, false);
-  }
-#endif
 }
 
 void PrimaryAccountManager::FirePrimaryAccountChanged(
