diff --git a/components/signin/internal/identity_manager/primary_account_mutator_impl.cc b/components/signin/internal/identity_manager/primary_account_mutator_impl.cc
index 65c261aac3..146b1311b8 100644
--- a/components/signin/internal/identity_manager/primary_account_mutator_impl.cc
+++ b/components/signin/internal/identity_manager/primary_account_mutator_impl.cc
@@ -17,7 +17,6 @@
 #include "components/signin/public/base/signin_buildflags.h"
 #include "components/signin/public/base/signin_client.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "google_apis/gaia/core_account_id.h"
 
@@ -57,7 +56,7 @@ PrimaryAccountMutatorImpl::SetPrimaryAccount(
   DCHECK(!account_info.gaia.empty());
 
 #if !BUILDFLAG(IS_CHROMEOS)
-  bool is_signin_allowed = pref_service_->GetBoolean(prefs::kSigninAllowed);
+  bool is_signin_allowed = false;
   if (!is_signin_allowed) {
     return PrimaryAccountError::kSigninNotAllowed;
   }
