diff --git a/chrome/browser/ash/login/signin/device_id_browsertest.cc b/chrome/browser/ash/login/signin/device_id_browsertest.cc
index eb16b19c6c..cd297c4e13 100644
--- a/chrome/browser/ash/login/signin/device_id_browsertest.cc
+++ b/chrome/browser/ash/login/signin/device_id_browsertest.cc
@@ -32,7 +32,6 @@
 #include "chrome/test/base/in_process_browser_test.h"
 #include "chromeos/dbus/constants/dbus_paths.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/user_manager/known_user.h"
 #include "components/user_manager/user_manager.h"
 #include "content/public/test/browser_test.h"
@@ -301,15 +300,6 @@ IN_PROC_BROWSER_TEST_F(DeviceIDTest, PRE_Migration) {
   SignInOnline(FakeGaiaMixin::kFakeUserEmail, FakeGaiaMixin::kFakeUserPassword,
                kRefreshToken1, FakeGaiaMixin::kFakeUserGaiaId);
 
-  // Simulate user that has device ID saved only in preferences (pre-M44).
-  PrefService* prefs =
-      ProfileHelper::Get()
-          ->GetProfileByUser(user_manager::UserManager::Get()->GetActiveUser())
-          ->GetPrefs();
-  prefs->SetString(
-      prefs::kGoogleServicesSigninScopedDeviceId,
-      GetDeviceId(AccountId::FromUserEmail(FakeGaiaMixin::kFakeUserEmail)));
-
   // Can't use SetKnownUserDeviceId here, because it forbids changing a device
   // ID.
   user_manager::KnownUser known_user(g_browser_process->local_state());
@@ -338,13 +328,6 @@ IN_PROC_BROWSER_TEST_F(DeviceIDTest, PRE_LegacyUsers) {
   SignInOnline(FakeGaiaMixin::kFakeUserEmail, FakeGaiaMixin::kFakeUserPassword,
                kRefreshToken1, FakeGaiaMixin::kFakeUserGaiaId);
 
-  PrefService* prefs =
-      ProfileHelper::Get()
-          ->GetProfileByUser(user_manager::UserManager::Get()->GetActiveUser())
-          ->GetPrefs();
-  EXPECT_TRUE(
-      prefs->GetString(prefs::kGoogleServicesSigninScopedDeviceId).empty());
-
   // Can't use SetKnownUserDeviceId here, because it forbids changing a device
   // ID.
   user_manager::KnownUser known_user(g_browser_process->local_state());
