diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 99506e19a4..355c666104 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -125,7 +125,6 @@ source_set("request_code_constants") {
 static_library("common_lib") {
   visibility = [
     ":*",
-    "//chrome/common/win:eventlog_messages",
   ]
   sources = [
     "chrome_content_client.cc",
@@ -363,7 +362,6 @@ static_library("common_lib") {
       "conflicts/remote_module_watcher_win.cc",
       "conflicts/remote_module_watcher_win.h",
     ]
-    deps += [ "//chrome/common/win:eventlog_messages" ]
 
     public_deps += [
       "//components/dom_distiller/core",  # Needed by chrome_content_client.cc.
@@ -526,6 +524,7 @@ static_library("url_constants") {
     "//components/optimization_guide/optimization_guide_internals/webui:url_constants",
     "//components/password_manager/content/common",
     "//components/safe_browsing/core/common",
+    "//components/supervised_user/core/common:buildflags",
     "//device/vr/buildflags",
   ]
 
