diff --git a/chrome/common/chrome_switches.cc b/chrome/common/chrome_switches.cc
index 6297693612bf360dddad9a7116ca8b9c3f7f349e..a3f07307eb55a6dc09ca244a1bc71f8ae221e72c 100644
--- a/chrome/common/chrome_switches.cc
+++ b/chrome/common/chrome_switches.cc
@@ -657,6 +657,9 @@ const char kSSLVersionTLSv13[] = "tls1.3";
 // Starts the browser maximized, regardless of any previous settings.
 const char kStartMaximized[] = "start-maximized";
 
+// Start the browser window without activating it (start inactive/background).
+const char kStartInactive[] = "start-inactive";
+
 // Starts the stack sampling profiler in the child process.
 const char kStartStackProfiler[] = "start-stack-profiler";
 
@@ -1058,4 +1061,10 @@ const char kUserDataMigrated[] = "user-data-migrated";
 // ifdef at the bottom. The order should match the header.
 // -----------------------------------------------------------------------------
 
+// Command-line switch to provide a DNS over HTTPS (DoH) template. See the
+// matching declaration in chrome_switches.h. The value should be a single
+// template string, for example:
+// --doh-template="https://dns.example.net/dns-query{?dns}"
+const char kDohTemplate[] = "doh-template";
+
 }  // namespace switches
