diff --git a/chrome/browser/ui/webui/policy/policy_ui_handler.cc b/chrome/browser/ui/webui/policy/policy_ui_handler.cc
index b41f2b75cf..7221bb7315 100644
--- a/chrome/browser/ui/webui/policy/policy_ui_handler.cc
+++ b/chrome/browser/ui/webui/policy/policy_ui_handler.cc
@@ -83,7 +83,6 @@
 #include "components/policy/proto/device_management_backend.pb.h"
 #include "components/prefs/pref_change_registrar.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/strings/grit/components_strings.h"
 #include "content/public/browser/browser_task_traits.h"
@@ -377,15 +376,6 @@ void PolicyUIHandler::HandleSetLocalTestPolicies(
 
   CHECK(local_test_provider);
 
-#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_CHROMEOS)
-  const std::string& profile_separation_policy_response = args[2].GetString();
-  Profile::FromWebUI(web_ui())->GetPrefs()->ClearPref(
-      prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage);
-  Profile::FromWebUI(web_ui())->GetPrefs()->SetDefaultPrefValue(
-      prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage,
-      base::Value(profile_separation_policy_response));
-#endif
-
   Profile::FromWebUI(web_ui())
       ->GetProfilePolicyConnector()
       ->UseLocalTestPolicyProvider();
@@ -399,13 +389,6 @@ void PolicyUIHandler::HandleRevertLocalTestPolicies(
   if (!PolicyUI::ShouldLoadTestPage(Profile::FromWebUI(web_ui()))) {
     return;
   }
-#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_CHROMEOS)
-  Profile::FromWebUI(web_ui())->GetPrefs()->ClearPref(
-      prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage);
-  Profile::FromWebUI(web_ui())->GetPrefs()->SetDefaultPrefValue(
-      prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage,
-      base::Value(std::string()));
-#endif
   Profile::FromWebUI(web_ui())
       ->GetProfilePolicyConnector()
       ->RevertUseLocalTestPolicyProvider();
