diff --git a/base/version_info/version_info.cc b/base/version_info/version_info.cc
index f42ee4c51ef360c4a2d5ac55a490970220c99b4e..de10b353130d706a8a5c802b2d99c1e1fe32a908 100644
--- a/base/version_info/version_info.cc
+++ b/base/version_info/version_info.cc
@@ -7,6 +7,7 @@
 #include <string>
 
 #include "base/no_destructor.h"
+#include "base/command_line.h"
 #include "base/strings/strcat.h"
 #include "base/strings/string_number_conversions.h"
 #include "base/version.h"
@@ -22,6 +23,17 @@ std::string GetMajorVersionNumber() {
   return base::NumberToString(GetMajorVersionNumberAsInt());
 }
 
+std::string_view GetVersionNumber() {
+  static base::NoDestructor<std::string> version([] {
+    base::CommandLine* cmd = base::CommandLine::ForCurrentProcess();
+    if (cmd && cmd->HasSwitch("user-agent-version")) {
+      return cmd->GetSwitchValueASCII("user-agent-version");
+    }
+    return std::string(PRODUCT_VERSION);
+  }());
+  return *version;
+}
+
 const base::Version& GetVersion() {
   static const base::NoDestructor<base::Version> version(GetVersionNumber());
   return *version;
