diff --git a/components/password_manager/core/browser/store_metrics_reporter.cc b/components/password_manager/core/browser/store_metrics_reporter.cc
index a816b0e083..ac61151eda 100644
--- a/components/password_manager/core/browser/store_metrics_reporter.cc
+++ b/components/password_manager/core/browser/store_metrics_reporter.cc
@@ -31,7 +31,6 @@
 #include "components/password_manager/core/common/password_manager_pref_names.h"
 #include "components/prefs/pref_service.h"
 #include "components/safe_browsing/core/common/features.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "google_apis/gaia/gaia_auth_util.h"
 #include "google_apis/gaia/gaia_urls.h"
 
@@ -756,7 +755,7 @@ StoreMetricsReporter::StoreMetricsReporter(
   is_account_storage_enabled_ =
       features_util::IsAccountStorageEnabled(sync_service);
 
-  is_safe_browsing_enabled_ = safe_browsing::IsSafeBrowsingEnabled(*prefs_);
+  is_safe_browsing_enabled_ = false;
 
   if (settings) {
     // TODO(crbug.com/358998546): use PasswordManagerSettingsService here.
