diff --git a/components/safe_browsing/core/browser/db/hit_report.h b/components/safe_browsing/core/browser/db/hit_report.h
index 4e7d4c7acc..b530420d21 100644
--- a/components/safe_browsing/core/browser/db/hit_report.h
+++ b/components/safe_browsing/core/browser/db/hit_report.h
@@ -10,7 +10,6 @@
 #include <string>
 
 #include "components/safe_browsing/core/browser/db/v4_protocol_manager_util.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "url/gurl.h"
 
 namespace safe_browsing {
@@ -62,7 +61,6 @@ struct HitReport {
   SBThreatType threat_type;
   ThreatSource threat_source;
 
-  ExtendedReportingLevel extended_reporting_level;
   bool is_enhanced_protection = false;
   bool is_metrics_reporting_active;
 
