diff --git a/components/safe_browsing/core/browser/db/util.h b/components/safe_browsing/core/browser/db/util.h
index 5584f2167e..494b54cfa9 100644
--- a/components/safe_browsing/core/browser/db/util.h
+++ b/components/safe_browsing/core/browser/db/util.h
@@ -17,7 +17,6 @@
 #include "base/containers/flat_map.h"
 #include "base/trace_event/traced_value.h"
 #include "components/safe_browsing/core/browser/db/v4_protocol_manager_util.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 
 namespace safe_browsing {
 
