diff --git a/components/safe_browsing/core/browser/db/v4_protocol_manager_util.h b/components/safe_browsing/core/browser/db/v4_protocol_manager_util.h
index b13314bfd5..f1752d2d01 100644
--- a/components/safe_browsing/core/browser/db/v4_protocol_manager_util.h
+++ b/components/safe_browsing/core/browser/db/v4_protocol_manager_util.h
@@ -19,9 +19,10 @@
 #include <vector>
 
 #include "base/containers/flat_set.h"
+#include "base/files/file_path.h"
 #include "base/gtest_prod_util.h"
+#include "base/time/time.h"
 #include "components/safe_browsing/core/browser/db/safebrowsing.pb.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "url/gurl.h"
 
 namespace net {
@@ -87,7 +88,6 @@ V4ProtocolConfig GetV4ProtocolConfig(const std::string& client_name,
 std::string GetReportUrl(
     const V4ProtocolConfig& config,
     const std::string& method,
-    const ExtendedReportingLevel* reporting_level = nullptr,
     const bool is_enhanced_protection = false);
 
 // Used to specify the type of check to perform in CheckBrowseUrl function.
