diff --git a/components/safe_browsing/core/browser/db/v4_update_protocol_manager.h b/components/safe_browsing/core/browser/db/v4_update_protocol_manager.h
index 434b32abc4..f834b6fe75 100644
--- a/components/safe_browsing/core/browser/db/v4_update_protocol_manager.h
+++ b/components/safe_browsing/core/browser/db/v4_update_protocol_manager.h
@@ -24,7 +24,6 @@
 #include "components/safe_browsing/core/browser/db/util.h"
 #include "components/safe_browsing/core/browser/db/v4_protocol_manager_util.h"
 #include "components/safe_browsing/core/common/proto/webui.pb.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 
 class GURL;
 
@@ -42,9 +41,6 @@ namespace safe_browsing {
 using V4UpdateCallback =
     base::RepeatingCallback<void(std::unique_ptr<ParsedServerResponse>)>;
 
-using ExtendedReportingLevelCallback =
-    base::RepeatingCallback<ExtendedReportingLevel()>;
-
 class V4UpdateProtocolManager {
  public:
   V4UpdateProtocolManager(const V4UpdateProtocolManager&) = delete;
@@ -59,8 +55,7 @@ class V4UpdateProtocolManager {
   V4UpdateProtocolManager(
       scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory,
       const V4ProtocolConfig& config,
-      V4UpdateCallback update_callback,
-      ExtendedReportingLevelCallback extended_reporting_level_callback);
+      V4UpdateCallback update_callback);
 
   void OnURLLoaderComplete(std::unique_ptr<std::string> response_body);
 
@@ -185,8 +180,6 @@ class V4UpdateProtocolManager {
   // complete.
   base::OneShotTimer timeout_timer_;
 
-  ExtendedReportingLevelCallback extended_reporting_level_callback_;
-
   SEQUENCE_CHECKER(sequence_checker_);
 };
 
