diff --git a/chrome/browser/extensions/chrome_extensions_browser_client_non_android.cc b/chrome/browser/extensions/chrome_extensions_browser_client_non_android.cc
index 1e8977267a..f583bf1168 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client_non_android.cc
+++ b/chrome/browser/extensions/chrome_extensions_browser_client_non_android.cc
@@ -31,8 +31,6 @@ void ChromeExtensionsBrowserClient::Init() {
   // Must occur after g_browser_process is initialized.
   user_script_listener_ = std::make_unique<UserScriptListener>();
 
-  // Full safe browsing is supported so use the Chrome delegate.
-  safe_browsing_delegate_ = std::make_unique<ChromeSafeBrowsingDelegate>();
 }
 
 ProcessManagerDelegate*
