diff --git a/chrome/browser/extensions/chrome_content_verifier_delegate.cc b/chrome/browser/extensions/chrome_content_verifier_delegate.cc
index ac2545794a..679bbf640c 100644
--- a/chrome/browser/extensions/chrome_content_verifier_delegate.cc
+++ b/chrome/browser/extensions/chrome_content_verifier_delegate.cc
@@ -303,17 +303,7 @@ bool ChromeContentVerifierDelegate::IsFromWebstore(
   // between which extensions are considered in-store.
   // See https://crbug.com/766806 for details.
   if (!InstallVerifier::IsFromStore(extension, context_)) {
-    // It's possible that the webstore update url was overridden for testing
-    // so also consider extensions with the default (production) update url
-    // to be from the store as well. Therefore update URL is compared with
-    // |GetDefaultWebstoreUpdateUrl|, not the |GetWebstoreUpdateUrl| used by
-    // |IsWebstoreUpdateUrl|.
-    ExtensionManagement* extension_management =
-        ExtensionManagementFactory::GetForBrowserContext(context_);
-    if (extension_management->GetEffectiveUpdateURL(extension) !=
-        extension_urls::GetDefaultWebstoreUpdateUrl()) {
-      return false;
-    }
+    return false;
   }
   return true;
 }
