diff --git a/chrome/browser/extensions/external_component_loader.cc b/chrome/browser/extensions/external_component_loader.cc
index 1df6574525..45e91db141 100644
--- a/chrome/browser/extensions/external_component_loader.cc
+++ b/chrome/browser/extensions/external_component_loader.cc
@@ -38,9 +38,6 @@ ExternalComponentLoader::~ExternalComponentLoader() = default;
 void ExternalComponentLoader::StartLoading() {
   auto prefs = base::Value::Dict();
   // Skip in-app payments app on Android. crbug.com/409396604
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING) && !BUILDFLAG(IS_ANDROID)
-  AddExternalExtension(extension_misc::kInAppPaymentsSupportAppId, prefs);
-#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING) && !BUILDFLAG(IS_ANDROID)
 
 #if BUILDFLAG(IS_CHROMEOS)
   {
