diff --git a/chrome/browser/extensions/extension_migrator.h b/chrome/browser/extensions/extension_migrator.h
index e86b4c2c44..171e040085 100644
--- a/chrome/browser/extensions/extension_migrator.h
+++ b/chrome/browser/extensions/extension_migrator.h
@@ -10,6 +10,7 @@
 #include "base/memory/raw_ptr.h"
 #include "chrome/browser/extensions/external_loader.h"
 #include "extensions/buildflags/buildflags.h"
+#include "url/gurl.h"
 
 static_assert(BUILDFLAG(ENABLE_EXTENSIONS_CORE));
 
@@ -43,6 +44,7 @@ class ExtensionMigrator : public ExternalLoader {
   raw_ptr<Profile> profile_;
   const std::string old_id_;
   const std::string new_id_;
+  GURL chrome_blank_;
 };
 
 }  // namespace extensions
