diff --git a/chrome/browser/extensions/extension_allowlist.cc b/chrome/browser/extensions/extension_allowlist.cc
index 44dc2f7350..f00709ed84 100644
--- a/chrome/browser/extensions/extension_allowlist.cc
+++ b/chrome/browser/extensions/extension_allowlist.cc
@@ -11,7 +11,6 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/safe_browsing/safe_browsing_metrics_collector_factory.h"
 #include "components/safe_browsing/core/browser/safe_browsing_metrics_collector.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "extensions/browser/allowlist_state.h"
 #include "extensions/browser/extension_registrar.h"
 #include "extensions/browser/extension_registry.h"
@@ -74,10 +73,6 @@ ExtensionAllowlist::ExtensionAllowlist(Profile* profile)
   // Register to Enhanced Safe Browsing setting changes for allowlist
   // enforcements.
   pref_change_registrar_.Init(profile_->GetPrefs());
-  pref_change_registrar_.Add(
-      prefs::kSafeBrowsingEnhanced,
-      base::BindRepeating(&ExtensionAllowlist::OnSafeBrowsingEnhancedChanged,
-                          base::Unretained(this)));
 }
 
 ExtensionAllowlist::~ExtensionAllowlist() = default;
@@ -247,15 +242,8 @@ void ExtensionAllowlist::OnExtensionInstalled(const ExtensionId& extension_id,
 }
 
 void ExtensionAllowlist::SetAllowlistEnforcementFields() {
-  if (safe_browsing::IsEnhancedProtectionEnabled(*profile_->GetPrefs())) {
-    warnings_enabled_ = base::FeatureList::IsEnabled(
-        extensions_features::kSafeBrowsingCrxAllowlistShowWarnings);
-    should_auto_disable_extensions_ = base::FeatureList::IsEnabled(
-        extensions_features::kSafeBrowsingCrxAllowlistAutoDisable);
-  } else {
     warnings_enabled_ = false;
     should_auto_disable_extensions_ = false;
-  }
 }
 
 // `ApplyEnforcement` can be called when an extension becomes not allowlisted or
@@ -406,15 +394,6 @@ void ExtensionAllowlist::NotifyExtensionAllowlistWarningStateChanged(
 }
 
 void ExtensionAllowlist::ReportExtensionReEnabledEvent() {
-  auto* metrics_collector =
-      safe_browsing::SafeBrowsingMetricsCollectorFactory::GetForProfile(
-          profile_);
-  DCHECK(metrics_collector);
-  if (metrics_collector) {
-    metrics_collector->AddSafeBrowsingEventToPref(
-        safe_browsing::SafeBrowsingMetricsCollector::EventType::
-            NON_ALLOWLISTED_EXTENSION_RE_ENABLED);
-  }
 }
 
 }  // namespace extensions
