diff --git a/chrome/browser/extensions/extension_migrator.cc b/chrome/browser/extensions/extension_migrator.cc
index ad3b8d93b9..af60cf6c37 100644
--- a/chrome/browser/extensions/extension_migrator.cc
+++ b/chrome/browser/extensions/extension_migrator.cc
@@ -19,7 +19,9 @@ namespace extensions {
 ExtensionMigrator::ExtensionMigrator(Profile* profile,
                                      const std::string& old_id,
                                      const std::string& new_id)
-    : profile_(profile), old_id_(old_id), new_id_(new_id) {}
+    : profile_(profile), old_id_(old_id), new_id_(new_id) {
+    chrome_blank_ = GURL("chrome://blank/");
+}
 
 ExtensionMigrator::~ExtensionMigrator() = default;
 
@@ -31,7 +33,7 @@ void ExtensionMigrator::StartLoading() {
   if (should_have_extension) {
     base::Value::Dict entry;
     entry.Set(ExternalProviderImpl::kExternalUpdateUrl,
-              extension_urls::GetWebstoreUpdateUrl().spec());
+              chrome_blank_.spec());
 
     prefs.Set(new_id_, std::move(entry));
   }
