diff --git a/chrome/browser/extensions/extension_allowlist_factory.cc b/chrome/browser/extensions/extension_allowlist_factory.cc
index 80ee67c180..7f6e27b277 100644
--- a/chrome/browser/extensions/extension_allowlist_factory.cc
+++ b/chrome/browser/extensions/extension_allowlist_factory.cc
@@ -45,7 +45,6 @@ ExtensionAllowlistFactory::ExtensionAllowlistFactory()
   DependsOn(ExtensionPrefsFactory::GetInstance());
   DependsOn(ExtensionRegistrarFactory::GetInstance());
   DependsOn(ExtensionRegistryFactory::GetInstance());
-  DependsOn(safe_browsing::SafeBrowsingMetricsCollectorFactory::GetInstance());
 }
 
 ExtensionAllowlistFactory::~ExtensionAllowlistFactory() = default;
