diff --git a/services/passage_embeddings/BUILD.gn b/services/passage_embeddings/BUILD.gn
index 7128e22e29..44a6e05203 100644
--- a/services/passage_embeddings/BUILD.gn
+++ b/services/passage_embeddings/BUILD.gn
@@ -31,11 +31,11 @@ source_set("passage_embeddings") {
       "//third_party/tflite_support:tflite_support_proto",
     ]
     deps += [
-      "//components/optimization_guide:machine_learning_tflite_buildflags",
       "//components/optimization_guide/core/inference:op_resolver",
       "//services/on_device_model:ml_internal_buildflags",
     ]
   }
+  deps += [ "//components/optimization_guide:machine_learning_tflite_buildflags" ]
 
   if (enable_ml_internal) {
     deps += [ "//services/on_device_model/ml" ]
