diff --git a/services/passage_embeddings/passage_embeddings_service.cc b/services/passage_embeddings/passage_embeddings_service.cc
index aa6b21f27e..e7b5542223 100644
--- a/services/passage_embeddings/passage_embeddings_service.cc
+++ b/services/passage_embeddings/passage_embeddings_service.cc
@@ -23,7 +23,6 @@ PassageEmbeddingsService::~PassageEmbeddingsService() = default;
 
 #if BUILDFLAG(BUILD_WITH_TFLITE_LIB)
 void PassageEmbeddingsService::OnEmbedderDisconnect() {
-  embedder_.reset();
 }
 #endif
 
