diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index a764a94cc7..20f07705d5 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -629,11 +629,6 @@ int GetContentRestrictions(const BrowserWindowInterface* bwi) {
     CoreTabHelper* core_tab_helper =
         CoreTabHelper::FromWebContents(current_tab);
     content_restrictions = core_tab_helper->content_restrictions();
-    NavigationEntry* last_committed_entry =
-        current_tab->GetController().GetLastCommittedEntry();
-    if (!content::IsSavableURL(last_committed_entry->GetURL())) {
-      content_restrictions |= CONTENT_RESTRICTION_SAVE;
-    }
   }
   return content_restrictions;
 }
@@ -1992,8 +1987,7 @@ bool CanSavePage(const Browser* browser) {
       policy::DownloadRestriction::ALL_FILES) {
     return false;
   }
-  return !browser->is_type_devtools() &&
-         !(GetContentRestrictions(browser) & CONTENT_RESTRICTION_SAVE);
+  return true;
 }
 
 void Print(BrowserWindowInterface* bwi) {
