diff --git a/components/signin/public/base/signin_switches.cc b/components/signin/public/base/signin_switches.cc
index c226da64f0..608f894519 100644
--- a/components/signin/public/base/signin_switches.cc
+++ b/components/signin/public/base/signin_switches.cc
@@ -7,7 +7,6 @@
 #include "base/feature_list.h"
 #include "base/time/time.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 
 #if BUILDFLAG(IS_WIN)
 #include "base/win/windows_version.h"
@@ -131,11 +130,6 @@ BASE_FEATURE(kEnableBoundSessionCredentials,
 );
 
 bool IsBoundSessionCredentialsEnabled(const PrefService* profile_prefs) {
-  // Enterprise policy takes precedence over the feature value.
-  if (profile_prefs->HasPrefPath(prefs::kBoundSessionCredentialsEnabled)) {
-    return profile_prefs->GetBoolean(prefs::kBoundSessionCredentialsEnabled);
-  }
-
   return base::FeatureList::IsEnabled(kEnableBoundSessionCredentials);
 }
 
@@ -174,11 +168,6 @@ BASE_FEATURE(kEnableChromeRefreshTokenBinding,
 );
 
 bool IsChromeRefreshTokenBindingEnabled(const PrefService* profile_prefs) {
-  // Enterprise policy takes precedence over the feature value.
-  if (profile_prefs->HasPrefPath(prefs::kBoundSessionCredentialsEnabled)) {
-    return profile_prefs->GetBoolean(prefs::kBoundSessionCredentialsEnabled);
-  }
-
   return base::FeatureList::IsEnabled(kEnableChromeRefreshTokenBinding);
 }
 
