diff --git a/base/allocator/partition_allocator/partition_alloc.gni b/base/allocator/partition_allocator/partition_alloc.gni
index 8b306a5aa9..ec4a10e2c3 100644
--- a/base/allocator/partition_allocator/partition_alloc.gni
+++ b/base/allocator/partition_allocator/partition_alloc.gni
@@ -234,9 +234,6 @@ declare_args() {
   force_enable_raw_ptr_exclusion = false
 }
 
-assert(!enable_pointer_compression_support || glue_core_pools,
-       "Pointer compression relies on core pools being contiguous.")
-
 declare_args() {
   # We want to use RawPtrBackupRefImpl as the raw_ptr<> implementation
   # iff BRP support is enabled. However, for purpose of performance
