diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
index 044ccd3f5e..ced129ad3e 100644
--- a/chrome/browser/ui/webui/settings/people_handler.cc
+++ b/chrome/browser/ui/webui/settings/people_handler.cc
@@ -57,7 +57,6 @@
 #include "components/signin/core/browser/signin_error_controller.h"
 #include "components/signin/public/base/consent_level.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_prefs.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/account_info.h"
@@ -413,10 +412,6 @@ void PeopleHandler::OnJavascriptAllowed() {
   PrefService* prefs = profile_->GetPrefs();
   profile_pref_registrar_ = std::make_unique<PrefChangeRegistrar>();
   profile_pref_registrar_->Init(prefs);
-  profile_pref_registrar_->Add(
-      prefs::kSigninAllowed,
-      base::BindRepeating(&PeopleHandler::UpdateSyncStatus,
-                          base::Unretained(this)));
 #if BUILDFLAG(ENABLE_DICE_SUPPORT)
   SigninPrefs::ObserveSigninPrefsChanges(
       *profile_pref_registrar_,
