diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.h b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.h
index a5deba28a015785da7df3412588b8e98b0b50a8c..11061b7bfa0a9a2fe43967956bf76df67a24ce99 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.h
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.h
@@ -310,7 +310,8 @@ class MODULES_EXPORT BaseRenderingContext2D : public CanvasRenderingContext,
                         unsigned run_start,
                         unsigned run_end,
                         double* max_width = nullptr,
-                        const Font* cluster_font = nullptr);
+                        const Font* cluster_font = nullptr,
+                        bool apply_geometry_noise = false);
 
   void PutByteArray(const SkPixmap& source,
                     const gfx::Rect& source_rect,
