diff --git a/chrome/browser/ui/signin/cookie_clear_on_exit_migration_notice.cc b/chrome/browser/ui/signin/cookie_clear_on_exit_migration_notice.cc
index 91fb228005..ad453cdbb4 100644
--- a/chrome/browser/ui/signin/cookie_clear_on_exit_migration_notice.cc
+++ b/chrome/browser/ui/signin/cookie_clear_on_exit_migration_notice.cc
@@ -23,7 +23,6 @@
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/consent_level.h"
 #include "components/signin/public/base/signin_buildflags.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/strings/grit/components_strings.h"
 #include "ui/base/l10n/l10n_util.h"
@@ -67,7 +66,6 @@ void OpenCookieSettingsAndCloseDialog(Browser& browser,
 }
 
 bool SetCookieClearOnExitMigrationComplete(PrefService& prefs, bool can_close) {
-  prefs.SetBoolean(prefs::kCookieClearOnExitMigrationNoticeComplete, true);
   return can_close;
 }
 
@@ -78,11 +76,6 @@ bool SetCookieClearOnExitMigrationComplete(PrefService& prefs, bool can_close) {
 bool CanShowCookieClearOnExitMigrationNotice(const Browser& browser) {
 #if BUILDFLAG(ENABLE_DICE_SUPPORT)
   Profile* profile = browser.profile();
-  PrefService* prefs = profile->GetPrefs();
-
-  if (prefs->GetBoolean(prefs::kCookieClearOnExitMigrationNoticeComplete)) {
-    return false;
-  }
 
   if (CookieClearOnExitMigrationNoticeShowingUserData::HasForProfile(
           *profile)) {
@@ -96,10 +89,6 @@ bool CanShowCookieClearOnExitMigrationNotice(const Browser& browser) {
   // User has to be signed in with UNO (non-syncing).
   signin::IdentityManager* identity_manager =
       IdentityManagerFactory::GetForProfile(profile);
-  if (!prefs->GetBoolean(prefs::kExplicitBrowserSignin)) {
-    return false;
-  }
-
   if (identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync)) {
     return false;
   }
