diff --git a/components/embedder_support/switches.h b/components/embedder_support/switches.h
index 088dc9271b..63e54e10d2 100644
--- a/components/embedder_support/switches.h
+++ b/components/embedder_support/switches.h
@@ -20,8 +20,12 @@ extern const char kOriginTrialDisabledTokens[];
 extern const char kOriginTrialPublicKey[];
 extern const char kShortReportingDelay[];
 extern const char kUseMobileUserAgent[];
+// A string used to override the default user agent with a custom one.
 extern const char kUserAgent[];
 
+// A string used to override the default user agent version with a custom one.
+extern const char kUserAgentVersion[];
+
 }  // namespace embedder_support
 
 #endif  // COMPONENTS_EMBEDDER_SUPPORT_SWITCHES_H_
