diff --git a/components/embedder_support/switches.cc b/components/embedder_support/switches.cc
index af1185607e4e4b2ad469972a8c089fcf26cf7edc..9591182da14b4e46fd037035b4ab04087be70444 100644
--- a/components/embedder_support/switches.cc
+++ b/components/embedder_support/switches.cc
@@ -33,4 +33,7 @@ const char kUseMobileUserAgent[] = "use-mobile-user-agent";
 // A string used to override the default user agent with a custom one.
 const char kUserAgent[] = "user-agent";
 
+// A string used to override the default user agent version with a custom one.
+const char kUserAgentVersion[] = "user-agent-version";
+
 }  // namespace embedder_support
