diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 557eadb18229e4adb2f0092593877254e2f1fe7c..18e14843e2e08f13a44758be63f15e54160348b2 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -163,9 +163,7 @@ const blink::UserAgentBrandList GetUserAgentBrandList(
   bool parse_result = base::StringToInt(major_version, &major_version_number);
   DCHECK(parse_result);
   std::optional<std::string> brand;
-#if !BUILDFLAG(CHROMIUM_BRANDING)
   brand = version_info::GetProductName();
-#endif
 
   std::string brand_version =
       output_version_type == blink::UserAgentBrandVersionType::kFullVersion
