diff --git a/components/embedder_support/user_agent_utils.h b/components/embedder_support/user_agent_utils.h
index 39a9d069561c68faf56cb0a75f938b120305b17e..850691627fa39e47996990274f6377faadabe5e8 100644
--- a/components/embedder_support/user_agent_utils.h
+++ b/components/embedder_support/user_agent_utils.h
@@ -30,6 +30,17 @@ std::string GetProductAndVersion();
 // when it is valid, or std::nullopt if it is not valid.
 std::optional<std::string> GetUserAgentFromCommandLine();
 
+// Returns a user agent version string passed via the kUserAgentVersion 
+// command-line argument if it is valid. Returns nullopt otherwise.
+std::optional<std::string> GetUserAgentVersionFromCommandLine();
+
+// Returns version number, respecting command line override if present.
+std::string GetVersionNumber();
+
+// Returns major version number, respecting command line override if present.  
+std::string GetMajorVersionNumber();
+
+
 // Returns the full or "reduced" user agent string, depending on the Reduce
 // User-Agent reduction phase features.
 std::string GetUserAgent();
