diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 9fd2e83b2a..c69376bb31 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -80,32 +80,6 @@ if (is_win && enable_resource_allowlist_generation) {
   _chrome_resource_allowlist = "$target_gen_dir/chrome_resource_allowlist.txt"
 }
 
-if (is_win) {
-  action("reorder_imports") {
-    script = "//build/win/reorder-imports.py"
-
-    # initialexe/ is used so that the we can reorder imports and write back to
-    # the final destination at $root_out_dir/.
-    inputs = [
-      "$root_out_dir/initialexe/chrome.exe",
-      "$root_out_dir/initialexe/chrome.exe.pdb",
-    ]
-    outputs = [
-      "$root_out_dir/chrome.exe",
-      "$root_out_dir/chrome.exe.pdb",
-    ]
-    args = [
-      "-i",
-      rebase_path("$root_out_dir/initialexe", root_build_dir),
-      "-o",
-      rebase_path("$root_out_dir", root_build_dir),
-      "-a",
-      current_cpu,
-    ]
-    deps = [ ":chrome_initial" ]
-  }
-}
-
 # This does not currently work. See crbug.com/1311822.
 # This target exists above chrome and it's main components in the dependency
 # tree as a central place to put assert_no_deps annotations. Since this depends
@@ -143,17 +117,9 @@ if (!is_android && !is_mac) {
     # Windows-only deps are OK because chrome_initial uses initialexe/chrome as
     # the output name for that platform.
     # See crbug.com/1146571.
-    if (is_win) {
-      public_deps += [ ":reorder_imports" ]
-      data_deps += [ ":reorder_imports" ]
-    }
   }
 
-  if (is_win) {
-    _chrome_output_name = "initialexe/chrome"
-  } else {
-    _chrome_output_name = "chrome"
-  }
+  _chrome_output_name = "chrome"
 
   executable("chrome_initial") {
     configs -= [ "//build/config/compiler:thinlto_optimize_default" ]
@@ -367,8 +333,6 @@ if (!is_android && !is_mac) {
     }
 
     data_deps += [
-      "//chrome/browser/resources/media/mei_preload:component",
-      "//components/privacy_sandbox/privacy_sandbox_attestations/preload:component",
       "//components/webapps/isolated_web_apps/preload:component",
       "//third_party/widevine/cdm",
     ]
@@ -937,8 +901,6 @@ if (is_win) {
 
   bundle_data("chrome_framework_resources") {
     sources = [
-      "//ui/gl/resources/angle-metal/gpu_shader_cache.bin",
-
       # This image is used to badge the lock icon in the
       # authentication dialogs, such as those used for installation
       # from disk image and Keystone promotion (if so enabled).  It
@@ -1184,8 +1146,6 @@ if (is_win) {
       ":optimization_guide_library",
       ":swiftshader_binaries",
       ":widevine_cdm_library",
-      "//chrome/browser/resources/media/mei_preload:component_bundle",
-      "//components/privacy_sandbox/privacy_sandbox_attestations/preload:component_bundle",
       "//components/webapps/isolated_web_apps/preload:component_bundle",
     ]
 
