diff --git a/chrome/browser/permissions/BUILD.gn b/chrome/browser/permissions/BUILD.gn
index 6f9bc905cc..2923e21670 100644
--- a/chrome/browser/permissions/BUILD.gn
+++ b/chrome/browser/permissions/BUILD.gn
@@ -44,10 +44,6 @@ source_set("permissions") {
     "prediction_service/passage_embedder_delegate.h",
     "prediction_service/permissions_aiv1_handler.cc",
     "prediction_service/permissions_aiv1_handler.h",
-    "prediction_service/prediction_based_permission_ui_selector.cc",
-    "prediction_service/prediction_based_permission_ui_selector.h",
-    "prediction_service/prediction_model_handler_provider.cc",
-    "prediction_service/prediction_model_handler_provider.h",
     "prediction_service/prediction_service_factory.cc",
     "prediction_service/prediction_service_factory.h",
     "prediction_service/prediction_service_request.cc",
