diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
index b45a26e611..ddf7bf701d 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
@@ -96,7 +96,7 @@ export class SettingsPrivacyPageElement extends SettingsPrivacyPageElementBase {
       enableSafeBrowsingSubresourceFilter_: {
         type: Boolean,
         value() {
-          return loadTimeData.getBoolean('enableSafeBrowsingSubresourceFilter');
+          return false;
         },
       },
 
