diff --git a/chrome/browser/resources/settings/privacy_page/security_page.ts b/chrome/browser/resources/settings/privacy_page/security_page.ts
index b65040b690..3c2bbbc3b0 100644
--- a/chrome/browser/resources/settings/privacy_page/security_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/security_page.ts
@@ -261,17 +261,6 @@ export class SettingsSecurityPageElement extends
     super.ready();
 
     CrSettingsPrefs.initialized.then(() => {
-      // Expand initial pref value manually because automatic
-      // expanding is disabled.
-      const prefValue = this.getPref('generated.safe_browsing').value;
-      if (prefValue === SafeBrowsingSetting.ENHANCED) {
-        this.$.safeBrowsingEnhanced.expanded = true;
-      } else if (prefValue === SafeBrowsingSetting.STANDARD) {
-        this.$.safeBrowsingStandard.expanded = true;
-      }
-
-      this.safeBrowsingStateOnOpen_ = prefValue;
-
       // The HTTPS-First Mode generated pref should never be set to
       // ENABLED_BALANCED if the feature flag is not enabled.
       if (!loadTimeData.getBoolean('enableHttpsFirstModeNewSettings')) {
@@ -281,10 +270,6 @@ export class SettingsSecurityPageElement extends
       }
     });
 
-    this.registerHelpBubble(
-        'kEnhancedProtectionSettingElementId',
-        this.$.safeBrowsingEnhanced.getBubbleAnchor(), {anchorPaddingTop: 10});
-
     // Initialize the last focus time on page load.
     this.lastFocusTime_ = HatsBrowserProxyImpl.getInstance().now();
 
@@ -422,8 +407,7 @@ export class SettingsSecurityPageElement extends
   }
 
   private getDisabledExtendedSafeBrowsing_(): boolean {
-    return this.getPref('generated.safe_browsing').value !==
-        SafeBrowsingSetting.STANDARD;
+    return true;
   }
 
   private getSafeBrowsingStandardSubLabel_(): string {
@@ -435,19 +419,6 @@ export class SettingsSecurityPageElement extends
 
   private getPasswordsLeakToggleSubLabel_(): string {
     let subLabel = this.i18n('passwordsLeakDetectionGeneralDescription');
-    // If the backing password leak detection preference is enabled, but the
-    // generated preference is off and user control is disabled, then additional
-    // text explaining that the feature will be enabled if the user signs in is
-    // added.
-    if (this.prefs !== undefined) {
-      const generatedPref = this.getPref('generated.password_leak_detection');
-      if (this.getPref('profile.password_manager_leak_detection').value &&
-          !generatedPref.value && generatedPref.userControlDisabled) {
-        subLabel +=
-            ' ' +  // Whitespace is a valid sentence separator w.r.t. i18n.
-            this.i18n('passwordsLeakDetectionSignedOutEnabledDescription');
-      }
-    }
     return subLabel;
   }
 
