diff --git a/chrome/browser/ui/webui/signin/signout_confirmation/signout_confirmation_handler.cc b/chrome/browser/ui/webui/signin/signout_confirmation/signout_confirmation_handler.cc
index f167afbe0a..c207124151 100644
--- a/chrome/browser/ui/webui/signin/signout_confirmation/signout_confirmation_handler.cc
+++ b/chrome/browser/ui/webui/signin/signout_confirmation/signout_confirmation_handler.cc
@@ -13,7 +13,6 @@
 #include "chrome/browser/ui/signin/signin_view_controller.h"
 #include "chrome/grit/branded_strings.h"
 #include "chrome/grit/generated_resources.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/strings/grit/components_strings.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "url/gurl.h"
