diff --git a/chrome/browser/resources/settings/autofill_page/payments_section.ts b/chrome/browser/resources/settings/autofill_page/payments_section.ts
index 08cf8b3014..93b30b1233 100644
--- a/chrome/browser/resources/settings/autofill_page/payments_section.ts
+++ b/chrome/browser/resources/settings/autofill_page/payments_section.ts
@@ -286,14 +286,6 @@ export class SettingsPaymentsSectionElement extends
     // Record that the user opened the payments settings.
     chrome.metricsPrivate.recordUserAction('AutofillCreditCardsViewed');
 
-    // Measure clicks on the 'Google Account' link for managing payment methods.
-    const manageAccountAnchor = this.$.manageLink.querySelector('a');
-    if (manageAccountAnchor !== null) {
-      manageAccountAnchor.addEventListener('click', () => {
-        MetricsBrowserProxyImpl.getInstance().recordAction(
-            'Autofill.PaymentMethodsSettingsPage.ManagePaymentMethodsLinkClicked');
-      });
-    }
   }
 
   override disconnectedCallback() {
