diff --git a/content/browser/interest_group/ad_auction_service_impl.cc b/content/browser/interest_group/ad_auction_service_impl.cc
index 857bf52f73..3c32d3b81c 100644
--- a/content/browser/interest_group/ad_auction_service_impl.cc
+++ b/content/browser/interest_group/ad_auction_service_impl.cc
@@ -193,11 +193,6 @@ AdAuctionServiceImpl::BiddingAndAuctionDataConstructionState::
 void AdAuctionServiceImpl::CreateMojoService(
     RenderFrameHost* render_frame_host,
     mojo::PendingReceiver<blink::mojom::AdAuctionService> receiver) {
-  CHECK(render_frame_host);
-
-  // The object is bound to the lifetime of `render_frame_host` and the mojo
-  // connection. See DocumentService for details.
-  new AdAuctionServiceImpl(*render_frame_host, std::move(receiver));
 }
 
 void AdAuctionServiceImpl::JoinInterestGroup(
