diff --git a/components/feed/core/v2/feed_stream.cc b/components/feed/core/v2/feed_stream.cc
index 0a539ef1ae..f35568b9f7 100644
--- a/components/feed/core/v2/feed_stream.cc
+++ b/components/feed/core/v2/feed_stream.cc
@@ -65,7 +65,6 @@
 #include "components/feed/feed_feature_list.h"
 #include "components/offline_pages/task/closure_task.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "google_apis/gaia/gaia_id.h"
 
 namespace feed {
@@ -174,9 +173,6 @@ FeedStream::FeedStream(RefreshTaskScheduler* refresh_task_scheduler,
   snippets_enabled_by_dse_.Init(prefs::kEnableSnippetsByDse, profile_prefs,
                                 preference_change_callback);
   has_stored_data_.Init(feed::prefs::kHasStoredData, profile_prefs);
-  signin_allowed_.Init(
-      ::prefs::kSigninAllowed, profile_prefs,
-      base::BindRepeating(&FeedStream::ClearAll, GetWeakPtr()));
   web_feed_subscription_coordinator_ =
       std::make_unique<WebFeedSubscriptionCoordinator>(delegate, this);
 
