diff --git a/services/on_device_model/ml/BUILD.gn b/services/on_device_model/ml/BUILD.gn
index 1f9972ec21..ef6bb77602 100644
--- a/services/on_device_model/ml/BUILD.gn
+++ b/services/on_device_model/ml/BUILD.gn
@@ -66,7 +66,6 @@ if (use_blink || (is_ios && build_with_internal_optimization_guide)) {
       "//components/optimization_guide/core:features",
       "//components/translate/core/language_detection",
       "//services/on_device_model:backend_interfaces",
-      "//services/on_device_model/safety:safety_util",
       "//third_party/abseil-cpp:absl",
       "//third_party/angle:includes",
       "//third_party/dawn/src/dawn:proc",
@@ -79,9 +78,6 @@ if (use_blink || (is_ios && build_with_internal_optimization_guide)) {
     if (use_blink) {
       deps += [ "//gpu/config" ]
     }
-    if (!is_fuchsia) {
-      deps += [ "//services/on_device_model/safety:bert_safety_model" ]
-    }
     if (is_win || is_mac || is_linux) {
       deps += [ "//third_party/xnnpack" ]
     }
