diff --git a/components/enterprise/browser/reporting/report_util.cc b/components/enterprise/browser/reporting/report_util.cc
index a1b0cd90a3..305b40241b 100644
--- a/components/enterprise/browser/reporting/report_util.cc
+++ b/components/enterprise/browser/reporting/report_util.cc
@@ -79,25 +79,6 @@ em::SettingValue TranslateSettingValue(
   }
 }
 
-em::ProfileSignalsReport::PasswordProtectionTrigger
-TranslatePasswordProtectionTrigger(
-    std::optional<safe_browsing::PasswordProtectionTrigger> trigger) {
-  if (trigger == std::nullopt) {
-    return em::ProfileSignalsReport::POLICY_UNSET;
-  }
-  switch (trigger.value()) {
-    case safe_browsing::PasswordProtectionTrigger::PASSWORD_PROTECTION_OFF:
-      return em::ProfileSignalsReport::PASSWORD_PROTECTION_OFF;
-    case safe_browsing::PasswordProtectionTrigger::PASSWORD_REUSE:
-      return em::ProfileSignalsReport::PASSWORD_REUSE;
-    case safe_browsing::PasswordProtectionTrigger::PHISHING_REUSE:
-      return em::ProfileSignalsReport::PHISHING_REUSE;
-    case safe_browsing::PasswordProtectionTrigger::
-        PASSWORD_PROTECTION_TRIGGER_MAX:
-      NOTREACHED();
-  }
-}
-
 em::ProfileSignalsReport::RealtimeUrlCheckMode TranslateRealtimeUrlCheckMode(
     enterprise_connectors::EnterpriseRealTimeUrlCheckMode mode) {
   switch (mode) {
@@ -110,18 +91,6 @@ em::ProfileSignalsReport::RealtimeUrlCheckMode TranslateRealtimeUrlCheckMode(
   }
 }
 
-em::ProfileSignalsReport::SafeBrowsingLevel TranslateSafeBrowsingLevel(
-    safe_browsing::SafeBrowsingState level) {
-  switch (level) {
-    case safe_browsing::SafeBrowsingState::NO_SAFE_BROWSING:
-      return em::ProfileSignalsReport::NO_SAFE_BROWSING;
-    case safe_browsing::SafeBrowsingState::STANDARD_PROTECTION:
-      return em::ProfileSignalsReport::STANDARD_PROTECTION;
-    case safe_browsing::SafeBrowsingState::ENHANCED_PROTECTION:
-      return em::ProfileSignalsReport::ENHANCED_PROTECTION;
-  }
-}
-
 #if BUILDFLAG(IS_WIN)
 std::unique_ptr<em::AntiVirusProduct> TranslateAvProduct(
     device_signals::AvProduct av_product) {
