diff --git a/components/enterprise/browser/reporting/report_util.h b/components/enterprise/browser/reporting/report_util.h
index d624683f2f..a6e7718524 100644
--- a/components/enterprise/browser/reporting/report_util.h
+++ b/components/enterprise/browser/reporting/report_util.h
@@ -23,17 +23,10 @@ std::string ObfuscateFilePath(const std::string& file_path);
 enterprise_management::SettingValue TranslateSettingValue(
     device_signals::SettingValue setting_value);
 
-enterprise_management::ProfileSignalsReport::PasswordProtectionTrigger
-TranslatePasswordProtectionTrigger(
-    std::optional<safe_browsing::PasswordProtectionTrigger> trigger);
-
 enterprise_management::ProfileSignalsReport::RealtimeUrlCheckMode
 TranslateRealtimeUrlCheckMode(
     enterprise_connectors::EnterpriseRealTimeUrlCheckMode mode);
 
-enterprise_management::ProfileSignalsReport::SafeBrowsingLevel
-TranslateSafeBrowsingLevel(safe_browsing::SafeBrowsingState level);
-
 #if BUILDFLAG(IS_WIN)
 std::unique_ptr<enterprise_management::AntiVirusProduct> TranslateAvProduct(
     device_signals::AvProduct av_product);
