diff --git a/chrome/browser/enterprise/connectors/connectors_service.cc b/chrome/browser/enterprise/connectors/connectors_service.cc
index 0a138ac0c7..049484c8ee 100644
--- a/chrome/browser/enterprise/connectors/connectors_service.cc
+++ b/chrome/browser/enterprise/connectors/connectors_service.cc
@@ -330,8 +330,7 @@ std::string ConnectorsService::GetManagementDomain() {
 
   std::optional<policy::PolicyScope> scope = std::nullopt;
   for (const char* scope_pref :
-       {enterprise_connectors::kEnterpriseRealTimeUrlCheckScope,
-        AnalysisConnectorScopePref(AnalysisConnector::FILE_ATTACHED),
+       {AnalysisConnectorScopePref(AnalysisConnector::FILE_ATTACHED),
         AnalysisConnectorScopePref(AnalysisConnector::FILE_DOWNLOADED),
         AnalysisConnectorScopePref(AnalysisConnector::BULK_DATA_ENTRY),
         AnalysisConnectorScopePref(AnalysisConnector::PRINT),
