diff --git a/chrome/browser/enterprise/connectors/referrer_cache_utils.cc b/chrome/browser/enterprise/connectors/referrer_cache_utils.cc
index 939d99c70f..c2234b00a1 100644
--- a/chrome/browser/enterprise/connectors/referrer_cache_utils.cc
+++ b/chrome/browser/enterprise/connectors/referrer_cache_utils.cc
@@ -44,11 +44,6 @@ safe_browsing::ReferrerChain GetSafeBrowsingReferrerChain(
     const GURL& url,
     content::WebContents& web_contents) {
   safe_browsing::ReferrerChain referrers;
-  safe_browsing::SafeBrowsingNavigationObserverManagerFactory::
-      GetForBrowserContext(web_contents.GetBrowserContext())
-          ->IdentifyReferrerChainByEventURL(
-              url, sessions::SessionTabHelper::IdForTab(&web_contents),
-              kReferrerUserGestureLimit, &referrers);
   return referrers;
 }
 
