diff --git a/components/safe_browsing/core/browser/realtime/chrome_enterprise_url_lookup_service.cc b/components/safe_browsing/core/browser/realtime/chrome_enterprise_url_lookup_service.cc
index 2172a168fd..f3661c1ba1 100644
--- a/components/safe_browsing/core/browser/realtime/chrome_enterprise_url_lookup_service.cc
+++ b/components/safe_browsing/core/browser/realtime/chrome_enterprise_url_lookup_service.cc
@@ -142,7 +142,7 @@ bool ChromeEnterpriseRealTimeUrlLookupService::
 
 bool ChromeEnterpriseRealTimeUrlLookupService::CanCheckSafeBrowsingDb() const {
   // Check database if safe browsing is enabled.
-  return safe_browsing::IsSafeBrowsingEnabled(*pref_service_);
+  return false;
 }
 
 bool ChromeEnterpriseRealTimeUrlLookupService::
