diff --git a/components/safe_browsing/core/browser/realtime/policy_engine.cc b/components/safe_browsing/core/browser/realtime/policy_engine.cc
index 985f922935..d8ce6abf17 100644
--- a/components/safe_browsing/core/browser/realtime/policy_engine.cc
+++ b/components/safe_browsing/core/browser/realtime/policy_engine.cc
@@ -11,7 +11,6 @@
 #include "components/enterprise/connectors/core/common.h"
 #include "components/enterprise/connectors/core/connectors_prefs.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/common/utils.h"
 #include "components/unified_consent/pref_names.h"
 #include "components/user_prefs/user_prefs.h"
@@ -56,7 +55,7 @@ bool RealTimePolicyEngine::IsUserMbbOptedIn(PrefService* pref_service) {
 
 // static
 bool RealTimePolicyEngine::IsUserEpOptedIn(PrefService* pref_service) {
-  return IsEnhancedProtectionEnabled(*pref_service);
+  return false;
 }
 
 // static
@@ -118,15 +117,7 @@ bool RealTimePolicyEngine::CanPerformEnterpriseFullURLLookup(
     return false;
   }
 
-  if (pref_service->GetInteger(
-          enterprise_connectors::kEnterpriseRealTimeUrlCheckMode) !=
-      enterprise_connectors::REAL_TIME_CHECK_FOR_MAINFRAME_ENABLED) {
-    base::UmaHistogramEnumeration("SafeBrowsing.RT.ConsumerVersionReason",
-                                  ConsumerVersionReason::POLICY_DISABLED);
-    return false;
-  }
-
-  return true;
+  return false;
 }
 
 }  // namespace safe_browsing
