diff --git a/chrome/browser/ui/webui/signin/turn_sync_on_helper_delegate_impl.cc b/chrome/browser/ui/webui/signin/turn_sync_on_helper_delegate_impl.cc
index c9d607835c..13b8506f82 100644
--- a/chrome/browser/ui/webui/signin/turn_sync_on_helper_delegate_impl.cc
+++ b/chrome/browser/ui/webui/signin/turn_sync_on_helper_delegate_impl.cc
@@ -41,7 +41,6 @@
 #include "components/policy/core/browser/signin/user_cloud_signin_restriction_policy_fetcher.h"
 #include "components/policy/core/common/policy_utils.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "google_apis/gaia/gaia_auth_util.h"
 #include "third_party/skia/include/core/SkColor.h"
 
@@ -253,26 +252,6 @@ void TurnSyncOnHelperDelegateImpl::OnProfileCheckComplete(
     return;
   }
 
-  if (prompt_for_new_profile) {
-    account_level_signin_restriction_policy_fetcher_
-        ->GetManagedAccountsSigninRestriction(
-            IdentityManagerFactory::GetForProfile(browser_->profile()),
-            account_info.account_id,
-            base::BindOnce(&TurnSyncOnHelperDelegateImpl::
-                               OnProfileSigninRestrictionsFetched,
-                           weak_ptr_factory_.GetWeakPtr(), account_info,
-                           std::move(callback)),
-            policy::utils::IsPolicyTestingEnabled(
-                browser_->profile()->GetPrefs(), chrome::GetChannel())
-                ? browser_->profile()
-                      ->GetPrefs()
-                      ->GetDefaultPrefValue(
-                          prefs::
-                              kUserCloudSigninPolicyResponseFromPolicyTestPage)
-                      ->GetString()
-                : std::string());
-    return;
-  }
 
   browser_->GetFeatures()
       .signin_view_controller()
