diff --git a/components/feed/core/v2/public/feed_service.cc b/components/feed/core/v2/public/feed_service.cc
index b5c2813804..08ea1c46ee 100644
--- a/components/feed/core/v2/public/feed_service.cc
+++ b/components/feed/core/v2/public/feed_service.cc
@@ -33,7 +33,6 @@
 #include "components/history/core/browser/history_types.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/consent_level.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/account_info.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "net/base/network_change_notifier.h"
@@ -183,7 +182,7 @@ class FeedService::StreamDelegateImpl : public FeedStream::Delegate {
   // Returns if signin is allowed on Android. Return true on other platform so
   // behavior is unchanged there.
   bool IsSigninAllowed() override {
-    return profile_prefs_->GetBoolean(::prefs::kSigninAllowed);
+    return false;
   }
   void RegisterExperiments(const Experiments& experiments) override {
     service_delegate_->RegisterExperiments(experiments);
