diff --git a/chrome/browser/accessibility/tree_fixing/BUILD.gn b/chrome/browser/accessibility/tree_fixing/BUILD.gn
index 77e43c52fd..c51ff55446 100644
--- a/chrome/browser/accessibility/tree_fixing/BUILD.gn
+++ b/chrome/browser/accessibility/tree_fixing/BUILD.gn
@@ -63,11 +63,9 @@ source_set("internal") {
   deps = [
     "//base",
     "//chrome/browser/profiles:profile",
-    "//chrome/browser/screen_ai:screen_ai_service_router_factory",
     "//components/paint_preview/browser",
     "//components/paint_preview/common",
     "//components/paint_preview/public",
     "//content/public/browser",
-    "//services/screen_ai/public/mojom",
   ]
 }
