diff --git a/components/omnibox/browser/featured_search_provider.cc b/components/omnibox/browser/featured_search_provider.cc
index 93a522ecf2..642eb6d0bc 100644
--- a/components/omnibox/browser/featured_search_provider.cc
+++ b/components/omnibox/browser/featured_search_provider.cc
@@ -290,20 +290,10 @@ void FeaturedSearchProvider::AddFeaturedKeywordMatches(
   for (TemplateURL* turl : turls) {
     if (turl->starter_pack_id() > 0 &&
         turl->is_active() == TemplateURLData::ActiveStatus::kTrue) {
-      // Skip @gemini if feature disabled.
-      if (turl->starter_pack_id() == template_url_starter_pack_data::kGemini &&
-          !OmniboxFieldTrial::IsStarterPackExpansionEnabled()) {
-        continue;
-      }
-      // Skip @page if feature disabled.
-      if (turl->starter_pack_id() == template_url_starter_pack_data::kPage &&
-          !omnibox_feature_configs::ContextualSearch::Get().starter_pack_page) {
-        continue;
-      }
-      // Skip @aimode if feature disabled.
-      if (turl->starter_pack_id() == template_url_starter_pack_data::kAiMode &&
-          !OmniboxFieldTrial::IsAimStarterPackEnabled(
-              client_->GetAimEligibilityService())) {
+      // Disable all Google AI starter pack options
+      if (turl->starter_pack_id() == template_url_starter_pack_data::kGemini ||
+          turl->starter_pack_id() == template_url_starter_pack_data::kAiMode ||
+          turl->starter_pack_id() == template_url_starter_pack_data::kPage) {
         continue;
       }
       // The history starter pack engine is disabled in incognito mode.
