diff --git a/chrome/browser/resources/settings/search_page/search_engine_edit_dialog.ts b/chrome/browser/resources/settings/search_page/search_engine_edit_dialog.ts
index 56e03c7e99..07f02b87fa 100644
--- a/chrome/browser/resources/settings/search_page/search_engine_edit_dialog.ts
+++ b/chrome/browser/resources/settings/search_page/search_engine_edit_dialog.ts
@@ -37,6 +37,7 @@ export interface SettingsSearchEngineEditDialogElement {
     dialog: CrDialogElement,
     keyword: CrInputElement,
     queryUrl: CrInputElement,
+    suggestionsUrl: CrInputElement,
     searchEngine: CrInputElement,
   };
 }
@@ -65,6 +66,7 @@ export class SettingsSearchEngineEditDialogElement extends
       searchEngine_: String,
       keyword_: String,
       queryUrl_: String,
+      suggestionsUrl_: String,
       dialogTitle_: String,
       actionButtonText_: String,
       showPolicySubtitle_: Boolean,
@@ -80,6 +82,7 @@ export class SettingsSearchEngineEditDialogElement extends
   declare private searchEngine_: string;
   declare private keyword_: string;
   declare private queryUrl_: string;
+  declare private suggestionsUrl_: string;
   declare private dialogTitle_: string;
   declare private actionButtonText_: string;
   declare private showPolicySubtitle_: boolean;
@@ -111,6 +114,7 @@ export class SettingsSearchEngineEditDialogElement extends
       this.searchEngine_ = this.model.name;
       this.keyword_ = this.model.keyword;
       this.queryUrl_ = this.model.url;
+      this.suggestionsUrl_ = this.model.suggestionsUrl;
     } else {
       this.dialogTitle_ = loadTimeData.getString('searchEnginesAddSiteSearch');
       this.actionButtonText_ = loadTimeData.getString('add');
@@ -147,8 +151,12 @@ export class SettingsSearchEngineEditDialogElement extends
       }
     }
 
-    [this.$.searchEngine, this.$.keyword, this.$.queryUrl].forEach(
-        element => this.validateElement_(element));
+    [
+      this.$.searchEngine,
+      this.$.keyword,
+      this.$.queryUrl,
+      this.$.suggestionsUrl
+    ].forEach(element => this.validateElement_(element));
   }
 
   private cancel_() {
@@ -157,7 +165,8 @@ export class SettingsSearchEngineEditDialogElement extends
 
   private onActionButtonClick_() {
     this.browserProxy_.searchEngineEditCompleted(
-        this.searchEngine_, this.keyword_, this.queryUrl_);
+        this.searchEngine_, this.keyword_, this.queryUrl_,
+        this.suggestionsUrl_);
     this.$.dialog.close();
   }
 
@@ -195,8 +204,10 @@ export class SettingsSearchEngineEditDialogElement extends
       this.$.searchEngine,
       this.$.keyword,
       this.$.queryUrl,
+      this.$.suggestionsUrl,
     ].every(function(inputElement) {
-      return !inputElement.invalid && inputElement.value.length > 0;
+      return !inputElement.invalid && (inputElement.value.length > 0 ||
+          inputElement.id == 'suggestionsUrl');
     });
     this.$.actionButton.disabled = !allValid;
   }
