diff --git a/chrome/browser/ui/pdf/chrome_pdf_document_helper_client.h b/chrome/browser/ui/pdf/chrome_pdf_document_helper_client.h
index 4a58023555..5fd39df1d4 100644
--- a/chrome/browser/ui/pdf/chrome_pdf_document_helper_client.h
+++ b/chrome/browser/ui/pdf/chrome_pdf_document_helper_client.h
@@ -25,7 +25,6 @@ class ChromePDFDocumentHelperClient : public pdf::PDFDocumentHelperClient {
   void OnSaveURL(content::WebContents* contents) override;
   void SetPluginCanSave(content::RenderFrameHost* render_frame_host,
                         bool can_save) override;
-  void OnSearchifyStarted(content::RenderFrameHost* render_frame_host) override;
 
   // Holds subscriptions for TabInterface callbacks.
   std::vector<base::CallbackListSubscription> tab_subscriptions_;
