diff --git a/chrome/browser/ui/webui/suggest_internals/suggest_internals_handler.cc b/chrome/browser/ui/webui/suggest_internals/suggest_internals_handler.cc
index 9549d9dfec..e846e76883 100644
--- a/chrome/browser/ui/webui/suggest_internals/suggest_internals_handler.cc
+++ b/chrome/browser/ui/webui/suggest_internals/suggest_internals_handler.cc
@@ -66,9 +66,6 @@ void SuggestInternalsHandler::OnRequestCreated(
       suggest_internals::mojom::Request::New();
   mojom_request->id = request_id;
   mojom_request->url = request->url;
-  std::string variations_header;
-  variations::GetVariationsHeader(*request, &variations_header);
-  mojom_request->data[variations::kClientDataHeader] = variations_header;
   mojom_request->data[request->method] = request->url.spec();
   mojom_request->status = suggest_internals::mojom::RequestStatus::kCreated;
   page_->OnRequestCreated(std::move(mojom_request));
