diff --git a/chrome/browser/ui/webui/extensions/extensions_ui_prefs.cc b/chrome/browser/ui/webui/extensions/extensions_ui_prefs.cc
index d95d8dfc35..92277c9378 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui_prefs.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui_prefs.cc
@@ -11,7 +11,7 @@ namespace extensions_ui_prefs {
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
-      prefs::kExtensionsUIDeveloperMode, false,
+    prefs::kExtensionsUIDeveloperMode, true,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 }
 
