diff --git a/chrome/browser/push_messaging/push_messaging_service_factory.cc b/chrome/browser/push_messaging/push_messaging_service_factory.cc
index 16b1dbbb87..9a0019fb59 100644
--- a/chrome/browser/push_messaging/push_messaging_service_factory.cc
+++ b/chrome/browser/push_messaging/push_messaging_service_factory.cc
@@ -19,7 +19,6 @@
 #include "chrome/browser/push_messaging/push_messaging_service_impl.h"
 #include "chrome/browser/safe_browsing/safe_browsing_service.h"
 #include "components/gcm_driver/instance_id/instance_id_profile_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 
 // static
 PushMessagingServiceImpl* PushMessagingServiceFactory::GetForProfile(
@@ -73,17 +72,7 @@ PushMessagingServiceFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   Profile* profile = Profile::FromBrowserContext(context);
   CHECK(!profile->IsOffTheRecord());
-  // Reporting service worker network requests should only be done for ESB
-  // users. The check below is the first ESB check. A second ESB check is
-  // performed before anything about the service worker is sent off device to
-  // Safe Browsing. If at the time of the second check the user is found to no
-  // longer be an ESB user, no Safe Browsing report will be sent.
-  bool includeSafeBrowsingDatabase =
-      g_browser_process && g_browser_process->safe_browsing_service() &&
-      safe_browsing::IsEnhancedProtectionEnabled(*profile->GetPrefs());
   return std::make_unique<PushMessagingServiceImpl>(
       profile,
-      includeSafeBrowsingDatabase
-          ? g_browser_process->safe_browsing_service()->database_manager()
-          : nullptr);
+      nullptr);
 }
