diff --git a/components/os_crypt/async/browser/dpapi_key_provider.cc b/components/os_crypt/async/browser/dpapi_key_provider.cc
index 2f0b6ee1f6..a8936746c8 100644
--- a/components/os_crypt/async/browser/dpapi_key_provider.cc
+++ b/components/os_crypt/async/browser/dpapi_key_provider.cc
@@ -14,6 +14,7 @@
 #include <wincrypt.h>
 
 #include "base/base64.h"
+#include "base/command_line.h"
 #include "base/logging.h"
 #include "base/metrics/histogram_functions.h"
 #include "base/notreached.h"
@@ -41,6 +42,9 @@ constexpr uint8_t kDPAPIKeyPrefix[] = {'D', 'P', 'A', 'P', 'I'};
 
 std::optional<std::vector<uint8_t>> DecryptKeyWithDPAPI(
     base::span<const uint8_t> ciphertext) {
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch("disable-encryption")) {
+    return std::vector<uint8_t>(ciphertext.begin(), ciphertext.end());
+  }
   DATA_BLOB input = {};
   input.pbData = const_cast<BYTE*>(ciphertext.data());
   input.cbData = static_cast<DWORD>(ciphertext.size());
