diff --git a/components/commerce/core/shopping_service.cc b/components/commerce/core/shopping_service.cc
index 952d6835f7..cf10d2aa92 100644
--- a/components/commerce/core/shopping_service.cc
+++ b/components/commerce/core/shopping_service.cc
@@ -62,7 +62,6 @@
 #include "components/search/ntp_features.h"
 #include "components/session_proto_db/session_proto_storage.h"
 #include "components/sessions/core/tab_restore_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/sync/base/features.h"
 #include "components/sync/service/sync_service.h"
 #include "components/unified_consent/url_keyed_data_collection_consent_helper.h"
@@ -164,16 +163,6 @@ class ProductSpecificationsUrlObserver
 // This function can be deleted once the Sync feature is removed.
 signin::ConsentLevel GetConsentLevelForEndpointFetchers(
     PrefService* pref_service) {
-  if (base::FeatureList::IsEnabled(
-          syncer::kReplaceSyncPromosWithSignInPromos)) {
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-    return pref_service->GetBoolean(prefs::kExplicitBrowserSignin)
-               ? signin::ConsentLevel::kSignin
-               : signin::ConsentLevel::kSync;
-#else
-    return signin::ConsentLevel::kSignin;
-#endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
-  }
   return signin::ConsentLevel::kSync;
 }
 
