diff --git a/chrome/browser/ui/webui/cr_components/history/history_util.cc b/chrome/browser/ui/webui/cr_components/history/history_util.cc
index cbf1fca421..3d0c726d46 100644
--- a/chrome/browser/ui/webui/cr_components/history/history_util.cc
+++ b/chrome/browser/ui/webui/cr_components/history/history_util.cc
@@ -20,7 +20,6 @@
 #include "components/history/core/common/pref_names.h"
 #include "components/history_clusters/core/history_clusters_prefs.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/strings/grit/components_strings.h"
 #include "content/public/browser/web_ui_data_source.h"
 #include "ui/base/l10n/l10n_util.h"
@@ -81,7 +80,7 @@ content::WebUIDataSource* HistoryUtil::PopulateCommonSourceForHistory(
 
   source->AddBoolean("isGuestSession", profile->IsGuestSession());
   source->AddBoolean("isSignInAllowed",
-                     prefs->GetBoolean(prefs::kSigninAllowed));
+                     false);
 
   source->AddBoolean(
       "enableBrowsingHistoryActorIntegrationM1",
