diff --git a/components/affiliations/core/browser/hash_affiliation_fetcher.cc b/components/affiliations/core/browser/hash_affiliation_fetcher.cc
index 572564dc03..fbb20aaf38 100644
--- a/components/affiliations/core/browser/hash_affiliation_fetcher.cc
+++ b/components/affiliations/core/browser/hash_affiliation_fetcher.cc
@@ -128,6 +128,10 @@ HashAffiliationFetcher::~HashAffiliationFetcher() {
 }
 
 void HashAffiliationFetcher::StartRequest(
+    const std::vector<FacetURI>& a, RequestInfo b,
+      base::OnceCallback<void(FetchResult)> c) {}
+[[maybe_unused]]
+void HashAffiliationFetcher::DeadStartRequest(
     const std::vector<FacetURI>& facet_uris,
     RequestInfo request_info,
     base::OnceCallback<void(FetchResult)> result_callback) {
