diff --git a/components/signin/public/identity_manager/identity_utils.cc b/components/signin/public/identity_manager/identity_utils.cc
index 0a747a7a5d..ddddbeb2de 100644
--- a/components/signin/public/identity_manager/identity_utils.cc
+++ b/components/signin/public/identity_manager/identity_utils.cc
@@ -14,7 +14,6 @@
 #include "base/strings/utf_string_conversions.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/consent_level.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/accounts_in_cookie_jar_info.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
@@ -66,21 +65,13 @@ bool IsUsernameAllowedByPattern(std::string_view username,
 
 bool IsUsernameAllowedByPatternFromPrefs(const PrefService* prefs,
                                          const std::string& username) {
-  return IsUsernameAllowedByPattern(
-      username, prefs->GetString(prefs::kGoogleServicesUsernamePattern));
+  return true;
 }
 
 bool IsImplicitBrowserSigninOrExplicitDisabled(
     const IdentityManager* identity_manager,
     const PrefService* prefs) {
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-  // Check if the user is implicitly signed in.
-  // Signed out users or signed in explicitly should return false.
-  return identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSignin) &&
-         !prefs->GetBoolean(prefs::kExplicitBrowserSignin);
-#else
-  return true;
-#endif
+  return false;
 }
 
 bool AreGoogleCookiesRebuiltAfterClearingWhenSignedIn(
