diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index eabe9386ea..a0e3d5a4a0 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1022,14 +1022,6 @@ int WebRequestEventRouter::OnBeforeRequest(
           DCHECK(action.redirect_url);
           OnDNRActionMatched(browser_context, *request, action);
           *new_url = GetNewUrl(action.redirect_url.value(), browser_context);
-          // Collect redirect action data for the Extension Telemetry Service.
-          if (action.type == DNRRequestAction::Type::REDIRECT) {
-            ExtensionsBrowserClient::Get()
-                ->GetSafeBrowsingDelegate()
-                ->NotifyExtensionDeclarativeNetRequestRedirectAction(
-                    browser_context, action.extension_id, request->url,
-                    action.redirect_url.value());
-          }
           RecordThatNavigationWasInitiatedByExtension(
               request, browser_context, new_url, action.extension_id);
           return net::OK;
