diff --git a/components/sync/service/sync_prefs.cc b/components/sync/service/sync_prefs.cc
index 342c0a2144..ba4894045e 100644
--- a/components/sync/service/sync_prefs.cc
+++ b/components/sync/service/sync_prefs.cc
@@ -25,8 +25,6 @@
 #include "components/saved_tab_groups/public/pref_names.h"
 #include "components/signin/public/base/gaia_id_hash.h"
 #include "components/signin/public/base/signin_buildflags.h"
-#include "components/signin/public/base/signin_pref_names.h"
-#include "components/signin/public/base/signin_prefs.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/sync/base/account_pref_utils.h"
 #include "components/sync/base/features.h"
@@ -129,18 +127,6 @@ SyncPrefs::SyncPrefs(PrefService* pref_service)
       base::BindRepeating(&SyncPrefs::OnSelectedTypesPrefChanged,
                           base::Unretained(this)));
 
-#if BUILDFLAG(ENABLE_DICE_SUPPORT)
-  if (base::FeatureList::IsEnabled(switches::kOfferMigrationToDiceUsers) ||
-      base::FeatureList::IsEnabled(switches::kRollbackDiceMigration)) {
-    // The explicit browser signin pref is used for determining whether some
-    // data types are selected by default. Therefore, upon a change, the
-    // selected types may change.
-    pref_change_registrar_.Add(
-        ::prefs::kExplicitBrowserSignin,
-        base::BindRepeating(&SyncPrefs::OnSelectedTypesPrefChanged,
-                            base::Unretained(this)));
-  }
-#endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
 }
 
 SyncPrefs::~SyncPrefs() {
@@ -245,7 +231,7 @@ bool SyncPrefs::IsExplicitBrowserSignin() const {
 #else
   // On desktop `prefs::kExplicitBrowserSignin` determines whether the sign-in
   // is explicit or implicit.
-  return pref_service_->GetBoolean(::prefs::kExplicitBrowserSignin);
+  return false;
 #endif
 }
 
@@ -1060,30 +1046,6 @@ void SyncPrefs::MaybeMigrateAutofillToPerAccountPref(
     return;
   }
   pref_service->SetBoolean(kAutofillPerAccountPrefMigrationDone, true);
-
-  const GaiaId last_syncing_gaia_id(
-      pref_service->GetString(::prefs::kGoogleServicesLastSyncingGaiaId));
-  if (last_syncing_gaia_id.empty()) {
-    return;
-  }
-
-  if (pref_service->GetBoolean(prefs::internal::kSyncKeepEverythingSynced)) {
-    return;
-  }
-
-  for (auto user_selectable_type :
-       {UserSelectableType::kPasswords, UserSelectableType::kAutofill}) {
-    const char* const pref_name_for_type =
-        GetPrefNameForType(user_selectable_type);
-    if (pref_service->GetBoolean(pref_name_for_type)) {
-      continue;
-    }
-
-    SetAccountKeyedPrefDictEntry(
-        pref_service, prefs::internal::kSelectedTypesPerAccount,
-        signin::GaiaIdHash::FromGaiaId(last_syncing_gaia_id),
-        pref_name_for_type, base::Value(false));
-  }
 }
 #endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
 
@@ -1124,21 +1086,15 @@ bool SyncPrefs::IsTypeSelectedByDefaultInTransportMode(
       // Before kReplaceSyncPromosWithSignInPromos, Bookmarks and Reading List
       // require a specific explicit sign in (relevant for desktop only).
       return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
-             SigninPrefs(*pref_service_)
-                 .GetBookmarksExplicitBrowserSignin(gaia_id) ||
              base::FeatureList::IsEnabled(
                  kEnableBookmarksSelectedTypeOnSigninForTesting);
     case UserSelectableType::kPreferences:
     case UserSelectableType::kThemes:
-      return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
-             pref_service_->GetBoolean(
-                 ::prefs::kPrefsThemesSearchEnginesAccountStorageEnabled);
+      return false;
     case UserSelectableType::kExtensions:
       // Before kReplaceSyncPromosWithSignInPromos, Extensions require a
       // specific explicit sign in.
-      return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
-             SigninPrefs(*pref_service_)
-                 .GetExtensionsExplicitBrowserSignin(gaia_id);
+      return false;
     case UserSelectableType::kApps:
     case UserSelectableType::kProductComparison:
     case UserSelectableType::kCookies:
