diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
index e6484141d7..eff462e981 100644
--- a/media/base/media_switches.cc
+++ b/media/base/media_switches.cc
@@ -773,7 +773,7 @@ BASE_FEATURE(OnDeviceWebSpeech,
 );
 
 // Enables the Live Caption feature on supported devices.
-BASE_FEATURE(LiveCaption, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(LiveCaption, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Logs a DumpWithoutCrashing() call each time the Speech On-Device API (SODA)
 // fails to load. Used to diagnose issues when rolling out new versions of the
@@ -1039,13 +1039,7 @@ BASE_FEATURE(UseAudioManagerMaxChannelLayout,
 #endif  // BUILDFLAG(IS_ANDROID)
 
 #if BUILDFLAG(ENABLE_HLS_DEMUXER)
-BASE_FEATURE(BuiltInHlsPlayer,
-#if BUILDFLAG(IS_ANDROID)
-             base::FEATURE_ENABLED_BY_DEFAULT
-#else
-             base::FEATURE_DISABLED_BY_DEFAULT
-#endif
-);
+BASE_FEATURE(BuiltInHlsPlayer, base::FEATURE_ENABLED_BY_DEFAULT);
 
 #endif  // BUILDFLAG(ENABLE_HLS_DEMUXER)
 
