diff --git a/chrome/browser/signin/account_consistency_mode_manager.cc b/chrome/browser/signin/account_consistency_mode_manager.cc
index 52a6425d14..007cfe68fb 100644
--- a/chrome/browser/signin/account_consistency_mode_manager.cc
+++ b/chrome/browser/signin/account_consistency_mode_manager.cc
@@ -24,7 +24,6 @@
 #include "components/pref_registry/pref_registry_syncable.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/signin_buildflags.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "google_apis/google_api_keys.h"
 
 #if BUILDFLAG(IS_CHROMEOS)
@@ -94,21 +93,6 @@ AccountConsistencyModeManager::AccountConsistencyModeManager(Profile* profile)
                           ->GetProfileAttributesStorage()
                           .GetProfileAttributesWithPath(profile_->GetPath())
                     : nullptr;
-  PrefService* prefs = profile_->GetPrefs();
-  // Propagate settings changes from the previous launch to the signin-allowed
-  // pref.
-  bool signin_allowed = IsDiceSignInAllowed(entry) &&
-                        prefs->GetBoolean(prefs::kSigninAllowedOnNextStartup);
-
-  // Disable sign-in if experimental-ai is enabled, regardless of channel.
-  auto* command_line = base::CommandLine::ForCurrentProcess();
-  if (command_line->HasSwitch(::switches::kExperimentalAiStableChannel)) {
-    signin_allowed = false;
-  }
-
-  prefs->SetBoolean(prefs::kSigninAllowed, signin_allowed);
-
-  UMA_HISTOGRAM_BOOLEAN("Signin.SigninAllowed", signin_allowed);
 #endif
 
   account_consistency_ = ComputeAccountConsistencyMethod(profile_);
@@ -121,7 +105,6 @@ AccountConsistencyModeManager::~AccountConsistencyModeManager() = default;
 // static
 void AccountConsistencyModeManager::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
-  registry->RegisterBooleanPref(prefs::kSigninAllowedOnNextStartup, true);
 }
 
 // static
@@ -203,13 +186,9 @@ AccountConsistencyModeManager::ComputeAccountConsistencyMethod(
 #if BUILDFLAG(ENABLE_MIRROR)
   return AccountConsistencyMethod::kMirror;
 #elif BUILDFLAG(ENABLE_DICE_SUPPORT)
-  if (!profile->GetPrefs()->GetBoolean(prefs::kSigninAllowed)) {
     VLOG(1) << "Desktop Identity Consistency disabled as sign-in to Chrome "
                "is not allowed";
     return AccountConsistencyMethod::kDisabled;
-  }
-
-  return AccountConsistencyMethod::kDice;
 #else
   NOTREACHED();
 #endif
