diff --git a/chrome/browser/signin/accounts_policy_manager.h b/chrome/browser/signin/accounts_policy_manager.h
index 2ddc4b5272..b87662ffa1 100644
--- a/chrome/browser/signin/accounts_policy_manager.h
+++ b/chrome/browser/signin/accounts_policy_manager.h
@@ -67,9 +67,6 @@ class AccountsPolicyManager : public KeyedService,
 
   raw_ptr<Profile> profile_;
 
-  // Helper object to listen for changes to the signin allowed preference.
-  BooleanPrefMember signin_allowed_;
-
   // Helper object to listen for changes to signin preferences stored in non-
   // profile-specific local prefs (like kGoogleServicesUsernamePattern).
   PrefChangeRegistrar local_state_pref_registrar_;
