diff --git a/chrome/browser/signin/accounts_policy_manager.cc b/chrome/browser/signin/accounts_policy_manager.cc
index 7e02b18a8c..7e2c6c81dc 100644
--- a/chrome/browser/signin/accounts_policy_manager.cc
+++ b/chrome/browser/signin/accounts_policy_manager.cc
@@ -31,7 +31,6 @@
 #include "components/policy/core/common/features.h"
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/account_info.h"
 #include "components/signin/public/identity_manager/accounts_mutator.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
@@ -190,25 +189,11 @@ void AccountsPolicyManager::Initialize() {
   EnsurePrimaryAccountAllowedForProfile(
       profile_, signin_metrics::ProfileSignout::kSigninNotAllowedOnProfileInit);
 
-  signin_allowed_.Init(
-      prefs::kSigninAllowed, profile_->GetPrefs(),
-      base::BindRepeating(&AccountsPolicyManager::OnSigninAllowedPrefChanged,
-                          weak_pointer_factory_.GetWeakPtr()));
-
   local_state_pref_registrar_.Init(g_browser_process->local_state());
-  local_state_pref_registrar_.Add(
-      prefs::kGoogleServicesUsernamePattern,
-      base::BindRepeating(
-          &AccountsPolicyManager::OnGoogleServicesUsernamePatternChanged,
-          weak_pointer_factory_.GetWeakPtr()));
 
   auto* identity_manager = IdentityManagerFactory::GetForProfile(profile_);
   identity_manager_observation_.Observe(identity_manager);
   profile_pref_change_registrar_.Init(profile_->GetPrefs());
-  profile_pref_change_registrar_.Add(
-      prefs::kProfileSeparationDomainExceptionList,
-      base::BindRepeating(&AccountsPolicyManager::RemoveUnallowedAccounts,
-                          weak_pointer_factory_.GetWeakPtr()));
   if (identity_manager->AreRefreshTokensLoaded()) {
     OnRefreshTokensLoaded();
   }
@@ -217,7 +202,6 @@ void AccountsPolicyManager::Initialize() {
 void AccountsPolicyManager::Shutdown() {
   profile_pref_change_registrar_.RemoveAll();
   local_state_pref_registrar_.RemoveAll();
-  signin_allowed_.Destroy();
 }
 
 void AccountsPolicyManager::OnGoogleServicesUsernamePatternChanged() {
@@ -241,11 +225,6 @@ void AccountsPolicyManager::EnsurePrimaryAccountAllowedForProfile(
 
   CoreAccountInfo primary_account =
       identity_manager->GetPrimaryAccountInfo(signin::ConsentLevel::kSync);
-  if (profile->GetPrefs()->GetBoolean(prefs::kSigninAllowed) &&
-      signin::IsUsernameAllowedByPatternFromPrefs(
-          g_browser_process->local_state(), primary_account.email)) {
-    return;
-  }
 
   if (ChromeSigninClientFactory::GetForProfile(profile)
           ->IsClearPrimaryAccountAllowed()) {
