diff --git a/chrome/browser/signin/chrome_device_id_helper.cc b/chrome/browser/signin/chrome_device_id_helper.cc
index c9b9bef5f1..7a656ca185 100644
--- a/chrome/browser/signin/chrome_device_id_helper.cc
+++ b/chrome/browser/signin/chrome_device_id_helper.cc
@@ -18,7 +18,6 @@
 #include "chrome/browser/ash/profiles/profile_helper.h"
 #include "chrome/browser/browser_process.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/user_manager/known_user.h"
 #include "components/user_manager/user_manager.h"
@@ -87,21 +86,11 @@ void MigrateSigninScopedDeviceId(Profile* profile) {
   user_manager::KnownUser known_user(g_browser_process->local_state());
   const AccountId account_id = user->GetAccountId();
   if (known_user.GetDeviceId(account_id).empty()) {
-    const std::string legacy_device_id = profile->GetPrefs()->GetString(
-        prefs::kGoogleServicesSigninScopedDeviceId);
-    if (!legacy_device_id.empty()) {
-      // Need to move device ID from the old location to the new one, if it has
-      // not been done yet.
-      known_user.SetDeviceId(account_id, legacy_device_id);
-    } else {
-      known_user.SetDeviceId(
-          account_id, GenerateSigninScopedDeviceId(
-                          user_manager::UserManager::Get()
-                              ->IsUserNonCryptohomeDataEphemeral(account_id)));
-    }
+    known_user.SetDeviceId(
+        account_id, GenerateSigninScopedDeviceId(
+                        user_manager::UserManager::Get()
+                            ->IsUserNonCryptohomeDataEphemeral(account_id)));
   }
-  profile->GetPrefs()->SetString(prefs::kGoogleServicesSigninScopedDeviceId,
-                                 std::string());
 }
 
 #endif  // BUILDFLAG(IS_CHROMEOS)
