diff --git a/chrome/browser/signin/dice_web_signin_interceptor.cc b/chrome/browser/signin/dice_web_signin_interceptor.cc
index ad6186f9d9..ed159465d8 100644
--- a/chrome/browser/signin/dice_web_signin_interceptor.cc
+++ b/chrome/browser/signin/dice_web_signin_interceptor.cc
@@ -74,7 +74,6 @@
 #include "components/search_engines/template_url_service.h"
 #include "components/signin/public/base/consent_level.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_prefs.h"
 #include "components/signin/public/identity_manager/account_capabilities.h"
 #include "components/signin/public/identity_manager/account_info.h"
@@ -463,16 +462,8 @@ void DiceWebSigninInterceptor::RegisterProfilePrefs(
   registry->RegisterBooleanPref(prefs::kSigninInterceptionEnabled, true);
   registry->RegisterStringPref(prefs::kManagedAccountsSigninRestriction,
                                std::string());
-  registry->RegisterStringPref(prefs::kSigninInterceptionIDPCookiesUrl,
-                               std::string());
   registry->RegisterBooleanPref(
       prefs::kManagedAccountsSigninRestrictionScopeMachine, false);
-  registry->RegisterIntegerPref(prefs::kProfileSeparationSettings, 0);
-  registry->RegisterIntegerPref(prefs::kProfileSeparationDataMigrationSettings,
-                                1);
-  registry->RegisterListPref(prefs::kProfileSeparationDomainExceptionList);
-  registry->RegisterStringPref(
-      prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage, std::string());
 }
 
 std::optional<SigninInterceptionHeuristicOutcome>
@@ -1532,16 +1523,6 @@ void DiceWebSigninInterceptor::
           g_browser_process->browser_policy_connector(),
           g_browser_process->system_network_context_manager()
               ->GetSharedURLLoaderFactory());
-  state_->account_level_signin_restriction_policy_fetcher_
-      ->GetManagedAccountsSigninRestriction(
-          identity_manager_, account_info.account_id, std::move(callback),
-          policy::utils::IsPolicyTestingEnabled(profile_->GetPrefs(),
-                                                chrome::GetChannel())
-              ? profile_->GetPrefs()
-                    ->GetDefaultPrefValue(
-                        prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage)
-                    ->GetString()
-              : std::string());
 }
 
 void DiceWebSigninInterceptor::
