diff --git a/chrome/browser/signin/signin_manager.cc b/chrome/browser/signin/signin_manager.cc
index 48d3fc6aec..3d47c0b1ba 100644
--- a/chrome/browser/signin/signin_manager.cc
+++ b/chrome/browser/signin/signin_manager.cc
@@ -16,7 +16,6 @@
 #include "components/prefs/pref_service.h"
 #include "components/signin/public/base/signin_client.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/accounts_in_cookie_jar_info.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
@@ -63,10 +62,6 @@ SigninManager::SigninManager(PrefService& prefs,
     : prefs_(prefs),
       signin_client_(client),
       identity_manager_(identity_manager) {
-  signin_allowed_.Init(
-      prefs::kSigninAllowed, &prefs,
-      base::BindRepeating(&SigninManager::OnSigninAllowedPrefChanged,
-                          base::Unretained(this)));
   UpdateUnconsentedPrimaryAccount();
   identity_manager_observation_.Observe(&identity_manager_.get());
 }
@@ -150,16 +145,6 @@ CoreAccountInfo SigninManager::ComputeUnconsentedPrimaryAccountInfo() const {
         signin::ConsentLevel::kSync);
   }
 
-  // Clearing the primary sync account when sign-in is not allowed is handled
-  // by PrimaryAccountPolicyManager. That flow is extremely hard to follow
-  // especially for the case when the user is syncing with a managed account
-  // as in that case the whole profile needs to be deleted.
-  //
-  // It was considered simpler to keep the logic to update the unconsented
-  // primary account in a single place.
-  if (!signin_allowed_.GetValue()) {
-    return CoreAccountInfo();
-  }
 
   bool is_current_primary_account_valid =
       IsValidUnconsentedPrimaryAccount(current_primary_account);
