diff --git a/net/tools/transport_security_state_generator/transport_security_state_generator.cc b/net/tools/transport_security_state_generator/transport_security_state_generator.cc
index 536bde8879..67100b9271 100644
--- a/net/tools/transport_security_state_generator/transport_security_state_generator.cc
+++ b/net/tools/transport_security_state_generator/transport_security_state_generator.cc
@@ -129,7 +129,7 @@ bool CheckDuplicateEntries(const TransportSecurityStateEntries& entries) {
 bool CheckNoopEntries(const TransportSecurityStateEntries& entries) {
   for (const auto& entry : entries) {
     if (!entry->force_https && entry->pinset.empty()) {
-      if (entry->hostname == "learn.doubleclick.net") {
+      if (entry->hostname == "learn.double" "click." "net") {
         // This entry is deliberately used as an exclusion.
         continue;
       }
