diff --git a/components/autofill/core/browser/data_manager/addresses/address_data_manager.cc b/components/autofill/core/browser/data_manager/addresses/address_data_manager.cc
index bffd49e47b..3c95241ff8 100644
--- a/components/autofill/core/browser/data_manager/addresses/address_data_manager.cc
+++ b/components/autofill/core/browser/data_manager/addresses/address_data_manager.cc
@@ -34,7 +34,6 @@
 #include "components/autofill/core/common/autofill_prefs.h"
 #include "components/autofill/core/common/dense_set.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/sync/base/data_type.h"
 #include "components/sync/base/features.h"
@@ -318,13 +317,7 @@ void AddressDataManager::RemoveLocalProfilesModifiedBetween(base::Time begin,
 }
 
 bool AddressDataManager::IsEligibleForAddressAccountStorage() const {
-  if (!sync_service_) {
     return false;
-  }
-
-  // The CONTACT_INFO data type is only running for eligible users. See
-  // ContactInfoDataTypeController.
-  return sync_service_->GetActiveDataTypes().Has(syncer::CONTACT_INFO);
 }
 
 void AddressDataManager::MigrateProfileToAccount(
@@ -608,35 +601,7 @@ bool AddressDataManager::IsAutofillUserSelectableTypeEnabled() const {
 }
 
 bool AddressDataManager::IsAutofillSyncToggleAvailable() const {
-  if (!pref_service_->GetBoolean(::prefs::kExplicitBrowserSignin)) {
-    return false;
-  }
-
-  if (!sync_service_) {
-    return false;
-  }
-
-  // Do not show the toggle if Sync is disabled on in error.
-  if (sync_service_->GetTransportState() ==
-          syncer::SyncService::TransportState::PAUSED ||
-      sync_service_->GetTransportState() ==
-          syncer::SyncService::TransportState::DISABLED) {
     return false;
-  }
-
-  // Do not show the toggle for syncing users.
-  if (sync_service_->HasSyncConsent()) {
-    return false;
-  }
-
-  if (sync_service_->GetUserSettings()->IsTypeManagedByPolicy(
-          syncer::UserSelectableType::kAutofill)) {
-    return false;
-  }
-
-  return contact_info_precondition_checker_ &&
-         contact_info_precondition_checker_->GetPreconditionState() ==
-             syncer::DataTypeController::PreconditionState::kPreconditionsMet;
 }
 
 void AddressDataManager::SetAutofillSelectableTypeEnabled(bool enabled) {
