diff --git a/chrome/browser/enterprise/signin/managed_profile_creation_controller.cc b/chrome/browser/enterprise/signin/managed_profile_creation_controller.cc
index 9f7667ee7e..e8242c4c29 100644
--- a/chrome/browser/enterprise/signin/managed_profile_creation_controller.cc
+++ b/chrome/browser/enterprise/signin/managed_profile_creation_controller.cc
@@ -32,7 +32,6 @@
 #include "components/policy/core/browser/signin/user_cloud_signin_restriction_policy_fetcher.h"
 #include "components/policy/core/common/policy_utils.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/accounts_mutator.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 
@@ -139,55 +138,6 @@ void ManagedProfileCreationController::OnProfileWillBeDestroyed(
 }
 
 void ManagedProfileCreationController::FetchProfileSeparationPolicies() {
-  // We should not fetch the policies twice.
-  CHECK(!policies_received_);
-  CHECK(!account_level_signin_restriction_policy_fetcher_);
-
-  auto policy_fetch_callback = base::BindOnce(
-      &ManagedProfileCreationController::OnProfileSeparationPoliciesReceived,
-      weak_ptr_factory_.GetWeakPtr());
-
-  if (profile_separation_policies_for_testing_.has_value()) {
-    CHECK_IS_TEST();
-    policy::ProfileSeparationPolicies profile_separation_policies =
-        std::exchange(profile_separation_policies_for_testing_, std::nullopt)
-            .value();
-    std::move(policy_fetch_callback)
-        .Run(std::move(profile_separation_policies));
-    return;
-  }
-
-  // If we cannot make network calls, we will not be able to fetch the
-  // account level policies.
-  if (!g_browser_process->system_network_context_manager()) {
-    std::move(policy_fetch_callback).Run(policy::ProfileSeparationPolicies());
-    return;
-  }
-
-  CHECK(source_profile_);
-  account_level_signin_restriction_policy_fetcher_ =
-      std::make_unique<policy::UserCloudSigninRestrictionPolicyFetcher>(
-          g_browser_process->browser_policy_connector(),
-          g_browser_process->system_network_context_manager()
-              ->GetSharedURLLoaderFactory());
-  account_level_signin_restriction_policy_fetcher_
-      ->GetManagedAccountsSigninRestriction(
-          GetIdentityManager(), account_info_.account_id,
-          std::move(policy_fetch_callback),
-          policy::utils::IsPolicyTestingEnabled(source_profile_->GetPrefs(),
-                                                chrome::GetChannel())
-              ? source_profile_->GetPrefs()
-                    ->GetDefaultPrefValue(
-                        prefs::kUserCloudSigninPolicyResponseFromPolicyTestPage)
-                    ->GetString()
-              : std::string());
-
-  policy_fetch_timeout_.Start(
-      FROM_HERE, base::Seconds(5),
-      base::BindOnce(&ManagedProfileCreationController::
-                         OnProfileSeparationPoliciesReceived,
-                     weak_ptr_factory_.GetWeakPtr(),
-                     policy::ProfileSeparationPolicies()));
 }
 
 void ManagedProfileCreationController::OnProfileSeparationPoliciesReceived(
