diff --git a/chrome/browser/enterprise/signin/oidc_authentication_signin_interceptor.cc b/chrome/browser/enterprise/signin/oidc_authentication_signin_interceptor.cc
index ad8c19b597..4dcc262020 100644
--- a/chrome/browser/enterprise/signin/oidc_authentication_signin_interceptor.cc
+++ b/chrome/browser/enterprise/signin/oidc_authentication_signin_interceptor.cc
@@ -49,7 +49,6 @@
 #include "components/policy/core/common/cloud/user_cloud_policy_manager.h"
 #include "components/policy/core/common/policy_logger.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/primary_account_mutator.h"
 #include "content/public/browser/storage_partition.h"
 #include "content/public/browser/web_contents.h"
@@ -437,8 +436,7 @@ void OidcAuthenticationSigninInterceptor::OnClientRegistered(
 
   // TODO(b/355270189): The interaction between OIDC profiles and BrowserSignin
   // policy should be finalized, this check only prevents Chrome from crashing.
-  if (dasher_based_ &&
-      !profile_->GetPrefs()->GetBoolean(prefs::kSigninAllowedOnNextStartup)) {
+  if (dasher_based_) {
     LOG_POLICY(ERROR, OIDC_ENROLLMENT)
         << "Google-synced OIDC profile can't be created because browser sign"
            "in is disabled.";
