diff --git a/chrome/browser/enterprise/signin/oidc_managed_profile_creation_delegate.cc b/chrome/browser/enterprise/signin/oidc_managed_profile_creation_delegate.cc
index b9b35c7933..d38b1b6bd0 100644
--- a/chrome/browser/enterprise/signin/oidc_managed_profile_creation_delegate.cc
+++ b/chrome/browser/enterprise/signin/oidc_managed_profile_creation_delegate.cc
@@ -9,7 +9,6 @@
 #include "chrome/browser/enterprise/signin/enterprise_signin_prefs.h"
 #include "chrome/browser/profiles/profile_attributes_storage.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 
 OidcManagedProfileCreationDelegate::OidcManagedProfileCreationDelegate() =
     default;
@@ -55,8 +54,6 @@ void OidcManagedProfileCreationDelegate::SetManagedAttributesForProfile(
 
 void OidcManagedProfileCreationDelegate::CheckManagedProfileStatus(
     Profile* new_profile) {
-  CHECK_EQ(new_profile->GetPrefs()->GetBoolean(prefs::kSigninAllowed),
-           dasher_based_);
 }
 
 void OidcManagedProfileCreationDelegate::OnManagedProfileInitialized(
