diff --git a/chrome/browser/enterprise/signin/profile_management_disclaimer_service.cc b/chrome/browser/enterprise/signin/profile_management_disclaimer_service.cc
index e0ae582130..d5d60e9344 100644
--- a/chrome/browser/enterprise/signin/profile_management_disclaimer_service.cc
+++ b/chrome/browser/enterprise/signin/profile_management_disclaimer_service.cc
@@ -39,7 +39,6 @@
 #include "components/policy/core/browser/signin/user_cloud_signin_restriction_policy_fetcher.h"
 #include "components/policy/core/common/policy_utils.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_prefs.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "components/signin/public/identity_manager/account_info.h"
