diff --git a/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.cc b/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.cc
index 4d646878d9..4b4242965a 100644
--- a/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.cc
+++ b/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.cc
@@ -7,7 +7,6 @@
 #include "chrome/browser/profiles/profile_attributes_storage.h"
 #include "chrome/browser/signin/signin_util.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 
 TokenManagedProfileCreationDelegate::TokenManagedProfileCreationDelegate() =
     default;
@@ -29,7 +28,6 @@ void TokenManagedProfileCreationDelegate::SetManagedAttributesForProfile(
 
 void TokenManagedProfileCreationDelegate::CheckManagedProfileStatus(
     Profile* new_profile) {
-  DCHECK(!new_profile->GetPrefs()->GetBoolean(prefs::kSigninAllowed));
 }
 
 void TokenManagedProfileCreationDelegate::OnManagedProfileInitialized(
