diff --git a/chrome/browser/ui/toolbar/app_menu_model.cc b/chrome/browser/ui/toolbar/app_menu_model.cc
index de6b83f162..af1cb0dffd 100644
--- a/chrome/browser/ui/toolbar/app_menu_model.cc
+++ b/chrome/browser/ui/toolbar/app_menu_model.cc
@@ -111,7 +111,6 @@
 #include "components/profile_metrics/browser_profile_type.h"
 #include "components/saved_tab_groups/public/features.h"
 #include "components/signin/public/base/signin_metrics.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/strings/grit/components_strings.h"
 #include "components/sync/base/features.h"
@@ -597,63 +596,7 @@ int ProfileSubMenuModel::GetAndIncrementNextMenuID() {
 }
 
 bool ProfileSubMenuModel::BuildSyncSection() {
-  if (!profile_->GetPrefs()->GetBoolean(prefs::kSigninAllowed)) {
     return false;
-  }
-
-  if (!SyncServiceFactory::IsSyncAllowed(profile_)) {
-    return false;
-  }
-
-  signin::IdentityManager* identity_manager =
-      IdentityManagerFactory::GetForProfile(profile_);
-  AddTitle(GetSyncSectionTitle(profile_, identity_manager));
-
-  // First, check for sync errors. They may exist even if sync-the-feature is
-  // disabled and only sync-the-transport is running.
-  const std::optional<AvatarSyncErrorType> error =
-      GetAvatarSyncErrorType(profile_);
-  if (error.has_value()) {
-    if (error == AvatarSyncErrorType::kSyncPaused) {
-      if (identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync)) {
-        // If sync is paused the menu item will be specific to the paused error.
-        AddItemWithStringIdAndVectorIcon(
-            this, IDC_SHOW_SIGNIN_WHEN_PAUSED, IDS_PROFILE_ROW_SIGN_IN_AGAIN,
-            vector_icons::kSyncOffChromeRefreshIcon);
-      } else {
-        AddItemWithStringIdAndVectorIcon(
-            this, IDC_SHOW_SIGNIN_WHEN_PAUSED,
-            IDS_PROFILES_VERIFY_ACCOUNT_BUTTON,
-            vector_icons::kAccountCircleOffChromeRefreshIcon);
-      }
-    } else {
-      // All remaining errors will have the same menu item.
-      AddItemWithStringIdAndVectorIcon(
-          this, IDC_SHOW_SYNC_SETTINGS, IDS_PROFILE_ROW_SYNC_ERROR_MESSAGE,
-          vector_icons::kSyncProblemChromeRefreshIcon);
-    }
-    return true;
-  }
-
-  if (identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync)) {
-    AddItemWithStringIdAndVectorIcon(this, IDC_SHOW_SYNC_SETTINGS,
-                                     IDS_PROFILE_ROW_SYNC_IS_ON,
-                                     vector_icons::kSyncChromeRefreshIcon);
-  } else {
-    if (base::FeatureList::IsEnabled(
-            syncer::kReplaceSyncPromosWithSignInPromos)) {
-      if (!identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSignin)) {
-        AddItemWithStringIdAndVectorIcon(this, IDC_SHOW_SIGNIN,
-                                         IDS_PROFILE_MENU_SIGNIN_PROMO_BUTTON,
-                                         vector_icons::kAccountCircleIcon);
-      }
-    } else {
-      AddItemWithStringIdAndVectorIcon(this, IDC_TURN_ON_SYNC,
-                                       IDS_PROFILE_ROW_TURN_ON_SYNC,
-                                       vector_icons::kSyncOffChromeRefreshIcon);
-    }
-  }
-  return true;
 }
 
 void ProfileSubMenuModel::BuildGuestProfileRow(Profile* profile) {
@@ -1017,16 +960,6 @@ void ExtensionsMenuModel::Build(Browser* browser) {
   SetElementIdentifierAt(
       GetIndexOfCommandId(IDC_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS).value(),
       kManageExtensionsMenuItem);
-  AddItemWithStringId(IDC_EXTENSIONS_SUBMENU_VISIT_CHROME_WEB_STORE,
-                      IDS_EXTENSIONS_SUBMENU_CHROME_WEBSTORE_ITEM);
-  SetElementIdentifierAt(
-      GetIndexOfCommandId(IDC_EXTENSIONS_SUBMENU_VISIT_CHROME_WEB_STORE)
-          .value(),
-      kVisitChromeWebStoreMenuItem);
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
-  SetCommandIcon(this, IDC_EXTENSIONS_SUBMENU_VISIT_CHROME_WEB_STORE,
-                 vector_icons::kGoogleChromeWebstoreIcon);
-#endif
 }
 
 ////////////////////////////////////////////////////////////////////////////////
