diff --git a/components/translate/core/browser/translate_manager.cc b/components/translate/core/browser/translate_manager.cc
index 689f467135..e7632ccb56 100644
--- a/components/translate/core/browser/translate_manager.cc
+++ b/components/translate/core/browser/translate_manager.cc
@@ -821,8 +821,9 @@ void TranslateManager::FilterIsTranslatePossible(
         TriggerDecision::kDisabledOffline);
   }
 
-  if (!ignore_missing_key_for_testing_ &&
-      !::google_apis::HasAPIKeyConfigured()) {
+  const base::CommandLine& command_line = *base::CommandLine::ForCurrentProcess();
+  if (!command_line.HasSwitch(translate::switches::kTranslateScriptURL) ||
+      (!ignore_missing_key_for_testing_ && !::google_apis::HasAPIKeyConfigured())) {
     // Without an API key, translate won't work, so don't offer to translate in
     // the first place. Leave kOfferTranslateEnabled on, though, because that
     // settings syncs and we don't want to turn off translate everywhere else.
