diff --git a/components/translate/core/browser/translate_script.cc b/components/translate/core/browser/translate_script.cc
index b7b4ca83d6..6237ba8205 100644
--- a/components/translate/core/browser/translate_script.cc
+++ b/components/translate/core/browser/translate_script.cc
@@ -146,8 +146,13 @@ void TranslateScript::OnScriptFetchComplete(bool success,
                         server_params.c_str());
 
     GURL security_origin = translate::GetTranslateSecurityOrigin();
-    base::StringAppendF(&data_, "var securityOrigin = '%s';\n",
-                        security_origin.spec().c_str());
+    const base::CommandLine& command_line = *base::CommandLine::ForCurrentProcess();
+    if (command_line.HasSwitch(translate::switches::kTranslateScriptURL)) {
+      base::StringAppendF(&data_, "var securityOrigin = '%s';\n",
+                          security_origin.spec().c_str());
+    } else {
+      base::StringAppendF(&data_, "var securityOrigin = '';\n");
+    }
 
     // Load embedded translate.js.
     data_.append(ui::ResourceBundle::GetSharedInstance().LoadDataResourceString(
