diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index db2b824216..74b2350cbf 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -297,7 +297,6 @@ source_set("impl") {
       "//chrome/browser/actor/ui",
       "//chrome/browser/contextual_cueing",
       "//chrome/browser/promos:utils",
-      "//chrome/browser/screen_ai:prefs",
       "//chrome/browser/search/background",
       "//chrome/browser/search_engine_choice",
       "//chrome/browser/search_engines",
