diff --git a/services/on_device_model/on_device_model.gni b/services/on_device_model/on_device_model.gni
index c1b55a7858..17717203e4 100644
--- a/services/on_device_model/on_device_model.gni
+++ b/services/on_device_model/on_device_model.gni
@@ -15,7 +15,6 @@ declare_args() {
   # Constraint code is disabled on platforms where the on-device model does not
   # run due to binary size.
   enable_constraints = is_win || is_mac || is_linux || is_ios || is_cbx
-  enable_ml_internal = build_with_internal_optimization_guide &&
-                       (is_win || is_mac || is_linux || is_ios || is_cbx)
+  enable_ml_internal = false
   use_chromeos_model_service = is_chrome_branded && is_cbx
 }
