diff --git a/components/safe_browsing/content/browser/triggers/suspicious_site_trigger_unittest.cc b/components/safe_browsing/content/browser/triggers/suspicious_site_trigger_unittest.cc
index 609946e620..858de87b32 100644
--- a/components/safe_browsing/content/browser/triggers/suspicious_site_trigger_unittest.cc
+++ b/components/safe_browsing/content/browser/triggers/suspicious_site_trigger_unittest.cc
@@ -12,7 +12,6 @@
 #include "build/build_config.h"
 #include "components/prefs/testing_pref_service.h"
 #include "components/safe_browsing/content/browser/triggers/mock_trigger_manager.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "content/public/test/browser_task_environment.h"
 #include "content/public/test/navigation_simulator.h"
 #include "content/public/test/test_renderer_host.h"
