diff --git a/components/safe_browsing/content/browser/triggers/trigger_throttler.cc b/components/safe_browsing/content/browser/triggers/trigger_throttler.cc
index 7474d4f168..e475e7fca0 100644
--- a/components/safe_browsing/content/browser/triggers/trigger_throttler.cc
+++ b/components/safe_browsing/content/browser/triggers/trigger_throttler.cc
@@ -146,30 +146,6 @@ void TriggerThrottler::CleanupOldEvents() {
 
 void TriggerThrottler::LoadTriggerEventsFromPref() {
   trigger_events_.clear();
-  if (!local_state_prefs_)
-    return;
-
-  const base::Value::Dict& event_dict =
-      local_state_prefs_->GetDict(prefs::kSafeBrowsingTriggerEventTimestamps);
-  for (auto trigger_pair : event_dict) {
-    // Check that the first item in the pair is convertible to a trigger type
-    // and that the second item is a list.
-    int trigger_type_int;
-    if (!base::StringToInt(trigger_pair.first, &trigger_type_int) ||
-        trigger_type_int < static_cast<int>(TriggerType::kMinTriggerType) ||
-        trigger_type_int > static_cast<int>(TriggerType::kMaxTriggerType)) {
-      continue;
-    }
-    if (!trigger_pair.second.is_list())
-      continue;
-
-    const TriggerType trigger_type = static_cast<TriggerType>(trigger_type_int);
-    for (const auto& timestamp : trigger_pair.second.GetList()) {
-      if (timestamp.is_double())
-        trigger_events_[trigger_type].push_back(
-            base::Time::FromSecondsSinceUnixEpoch(timestamp.GetDouble()));
-    }
-  }
 }
 
 void TriggerThrottler::WriteTriggerEventsToPref() {
@@ -186,9 +162,6 @@ void TriggerThrottler::WriteTriggerEventsToPref() {
     trigger_dict.Set(base::NumberToString(static_cast<int>(trigger_item.first)),
                      std::move(timestamps));
   }
-
-  local_state_prefs_->SetDict(prefs::kSafeBrowsingTriggerEventTimestamps,
-                              std::move(trigger_dict));
 }
 
 size_t TriggerThrottler::GetDailyQuotaForTrigger(
