diff --git a/content/browser/webid/request_service.cc b/content/browser/webid/request_service.cc
index bffa357116..9456952390 100644
--- a/content/browser/webid/request_service.cc
+++ b/content/browser/webid/request_service.cc
@@ -192,31 +192,6 @@ RequestService::~RequestService() {
 void RequestService::Create(
     RenderFrameHost* host,
     mojo::PendingReceiver<blink::mojom::FederatedAuthRequest> receiver) {
-  CHECK(host);
-
-  BrowserContext* browser_context = host->GetBrowserContext();
-  raw_ptr<FederatedIdentityApiPermissionContextDelegate>
-      api_permission_context =
-          browser_context->GetFederatedIdentityApiPermissionContext();
-  raw_ptr<FederatedIdentityAutoReauthnPermissionContextDelegate>
-      auto_reauthn_permission_context =
-          browser_context->GetFederatedIdentityAutoReauthnPermissionContext();
-  raw_ptr<FederatedIdentityPermissionContextDelegate> permission_context =
-      browser_context->GetFederatedIdentityPermissionContext();
-  raw_ptr<IdentityRegistry> identity_registry =
-      IdentityRegistry::FromWebContents(WebContents::FromRenderFrameHost(host));
-
-  if (!api_permission_context || !auto_reauthn_permission_context ||
-      !permission_context) {
-    return;
-  }
-
-  // RequestService owns itself. It will self-destruct when a mojo
-  // interface error occurs, the RenderFrameHost is deleted, or the
-  // RenderFrameHost navigates to a new document.
-  new RequestService(*host, api_permission_context,
-                     auto_reauthn_permission_context, permission_context,
-                     identity_registry, std::move(receiver));
 }
 
 RequestService& RequestService::CreateForTesting(
