diff --git a/chrome/browser/resources/extensions/item_list.html.ts b/chrome/browser/resources/extensions/item_list.html.ts
index ba7eac7d5c..b38bf0822e 100644
--- a/chrome/browser/resources/extensions/item_list.html.ts
+++ b/chrome/browser/resources/extensions/item_list.html.ts
@@ -30,12 +30,6 @@ export function getHtml(this: ExtensionsItemListElement) {
         </extensions-mv2-deprecation-panel>
       </div>` : ''}
 
-    <div id="no-items" class="empty-list-message"
-        ?hidden="${!this.shouldShowEmptyItemsMessage_()}">
-      <span @click="${this.onNoExtensionsClick_}">
-        $i18nRaw{noExtensionsOrApps}
-      </span>
-    </div>
     <div id="no-search-results" class="empty-list-message"
         ?hidden="${!this.shouldShowEmptySearchMessage_()}">
       <span>$i18n{noSearchResults}</span>
