diff --git a/chrome/browser/resources/extensions/sidebar.html.ts b/chrome/browser/resources/extensions/sidebar.html.ts
index c72603eca1..590d1718f2 100644
--- a/chrome/browser/resources/extensions/sidebar.html.ts
+++ b/chrome/browser/resources/extensions/sidebar.html.ts
@@ -33,24 +33,6 @@ export function getHtml(this: ExtensionsSidebarElement) {
     <cr-ripple></cr-ripple>
   </a>
 </cr-menu-selector>
-<div class="separator" ?hidden="${!this.inDevMode}"></div>
-      ${this.inDevMode ? html`
-        <div class="cr-nav-menu-item" id="moreExtensions">
-          <span id="promo-message-text" class="cr-secondary-text"
-            .innerHTML="${this.computeDocsPromoText_()}">
-          </span>
-        </div>
-        `: ''}
-<div class="separator"></div>
-<div class="cr-nav-menu-item" id="moreExtensions">
-  <cr-icon id="web-store-icon" icon="extensions-icons:web_store">
-  </cr-icon>
-  <span id="discover-more-text" class="cr-secondary-text"
-      @click="${this.onMoreExtensionsClick_}"
-      .innerHTML="${this.computeDiscoverMoreText_()}">
-  </span>
-  <cr-ripple></cr-ripple>
-</div>
 <!--_html_template_end_-->`;
   // clang-format on
 }
