diff --git a/chrome/browser/resources/extensions/sidebar.ts b/chrome/browser/resources/extensions/sidebar.ts
index 3f86428dc3..994874d85e 100644
--- a/chrome/browser/resources/extensions/sidebar.ts
+++ b/chrome/browser/resources/extensions/sidebar.ts
@@ -8,7 +8,6 @@ import './icons.html.js';
 import type {CrMenuSelector} from 'chrome://resources/cr_elements/cr_menu_selector/cr_menu_selector.js';
 import {I18nMixinLit} from 'chrome://resources/cr_elements/i18n_mixin_lit.js';
 import {assert} from 'chrome://resources/js/assert.js';
-import {loadTimeData} from 'chrome://resources/js/load_time_data.js';
 import {CrLitElement} from 'chrome://resources/lit/v3_0/lit.rollup.js';
 import type {PropertyValues} from 'chrome://resources/lit/v3_0/lit.rollup.js';
 
@@ -113,21 +112,6 @@ export class ExtensionsSidebarElement extends ExtensionsSidebarElementBase {
     }
   }
 
-  protected computeDiscoverMoreText_(): TrustedHTML {
-    return this.i18nAdvanced('sidebarDiscoverMore', {
-      tags: ['a'],
-      attrs: ['target'],
-      substitutions: [loadTimeData.getString('getMoreExtensionsUrl')],
-    });
-  }
-
-  protected computeDocsPromoText_(): TrustedHTML {
-    return this.i18nAdvanced('sidebarDocsPromo', {
-      tags: ['a'],
-      attrs: ['target'],
-      substitutions: [loadTimeData.getString('extensionsWhatsNewURL')],
-    });
-  }
 }
 
 declare global {
