diff --git a/ui/platform_window/win/win_window.cc b/ui/platform_window/win/win_window.cc
index 62a4d5bd3a..197059b3a6 100644
--- a/ui/platform_window/win/win_window.cc
+++ b/ui/platform_window/win/win_window.cc
@@ -69,7 +69,7 @@ void WinWindow::Destroy() {
 }
 
 void WinWindow::Show(bool inactive) {
-  ShowWindow(hwnd(), inactive ? SW_SHOWNOACTIVATE : SW_SHOWNORMAL);
+  ShowWindow(hwnd(), SW_SHOWNOACTIVATE);
 }
 
 void WinWindow::Hide() {
