diff --git a/ui/platform_window/win/win_window.cc b/ui/platform_window/win/win_window.cc
index 1c6b85b8ba45d76e36397603c6ffcd52aa88b633..5832ed7ac82e6b84b748a5e8b15c96860ef4b2ae 100644
--- a/ui/platform_window/win/win_window.cc
+++ b/ui/platform_window/win/win_window.cc
@@ -98,7 +98,7 @@ void WinWindow::OnCaretBoundsChanged(const TextInputClient* client) {}
 void WinWindow::OnTextInputStateChanged(const TextInputClient* client) {}
 
 void WinWindow::Show(bool inactive) {
-  ShowWindow(hwnd(), inactive ? SW_SHOWNOACTIVATE : SW_SHOWNORMAL);
+  ShowWindow(hwnd(), SW_SHOWNOACTIVATE);
 }
 
 void WinWindow::Hide() {
