diff --git a/chrome/browser/ui/views/user_education/browser_ntp_promos.cc b/chrome/browser/ui/views/user_education/browser_ntp_promos.cc
index 3dd1d582a6..807d6ddb74 100644
--- a/chrome/browser/ui/views/user_education/browser_ntp_promos.cc
+++ b/chrome/browser/ui/views/user_education/browser_ntp_promos.cc
@@ -22,7 +22,6 @@
 #include "chrome/browser/user_education/ntp_promo_identifiers.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/user_education/common/ntp_promo/ntp_promo_registry.h"
 #include "components/user_education/common/ntp_promo/ntp_promo_specification.h"
@@ -38,28 +37,7 @@ namespace {
 
 NtpPromoSpecification::Eligibility CheckSignInPromoEligibility(
     Profile* profile) {
-  if (!profile->GetPrefs()->GetBoolean(prefs::kSigninAllowed)) {
     return NtpPromoSpecification::Eligibility::kIneligible;
-  }
-
-  const auto signed_in_state = signin_util::GetSignedInState(
-      IdentityManagerFactory::GetForProfile(profile));
-  switch (signed_in_state) {
-    case signin_util::SignedInState::kSignedOut:
-      // User is fully signed out.
-      return NtpPromoSpecification::Eligibility::kEligible;
-    case signin_util::SignedInState::kWebOnlySignedIn:
-      // When signed in on the web, one-click sign in options exist elsewhere
-      // in Chrome. This promo currently only offers the full-sign-in flow, so
-      // don't show it to users already signed in on the Web.
-      return NtpPromoSpecification::Eligibility::kIneligible;
-    case signin_util::SignedInState::kSignedIn:
-    case signin_util::SignedInState::kSyncing:
-    case signin_util::SignedInState::kSignInPending:
-    case signin_util::SignedInState::kSyncPaused:
-      // All other cases are considered completed.
-      return NtpPromoSpecification::Eligibility::kCompleted;
-  }
 }
 
 void SignInPromoShown() {
