diff --git a/chrome/browser/enterprise/profile_management/profile_management_navigation_throttle.cc b/chrome/browser/enterprise/profile_management/profile_management_navigation_throttle.cc
index 5679065965..a843e959e1 100644
--- a/chrome/browser/enterprise/profile_management/profile_management_navigation_throttle.cc
+++ b/chrome/browser/enterprise/profile_management/profile_management_navigation_throttle.cc
@@ -26,7 +26,6 @@
 #include "chrome/common/chrome_switches.h"
 #include "components/account_id/account_id.h"
 #include "components/prefs/pref_service.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/navigation_throttle.h"
@@ -318,12 +317,6 @@ void ProfileManagementNavigationThrottle::RegisterWithDomain(
   std::optional<std::string> management_domain =
       GetDomainFromAttributeValue(domain);
   if (management_domain) {
-    auto* prefs =
-        Profile::FromBrowserContext(
-            navigation_handle()->GetWebContents()->GetBrowserContext())
-            ->GetPrefs();
-    prefs->SetString(prefs::kSigninInterceptionIDPCookiesUrl,
-                     navigation_handle()->GetURL().spec());
     PostNavigateTo(GURL(base::StringPrintf(kGoogleServiceLoginUrl,
                                            management_domain.value().c_str())));
     return;
@@ -345,11 +338,6 @@ void ProfileManagementNavigationThrottle::RegisterWithToken(
     PostNavigateTo(GURL(token_url_for_testing_));
     return;
   }
-  auto* prefs = Profile::FromBrowserContext(
-                    navigation_handle()->GetWebContents()->GetBrowserContext())
-                    ->GetPrefs();
-  prefs->SetString(prefs::kSigninInterceptionIDPCookiesUrl,
-                   navigation_handle()->GetURL().spec());
 
   auto* interceptor = ProfileTokenWebSigninInterceptorFactory::GetForProfile(
       Profile::FromBrowserContext(
