diff --git a/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc b/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
index 58f45bf5a6..e36b7698fd 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc
@@ -1241,12 +1241,14 @@ void BookmarkMenuDelegate::BuildOtherNodeMenuHeader(MenuItemView* menu) {
   menu->AppendMenuItem(
       IDC_SHOW_BOOKMARK_SIDE_PANEL,
       l10n_util::GetStringUTF16(IDS_BOOKMARKS_ALL_BOOKMARKS_OPEN_SIDE_PANEL),
-      bookmarks_side_panel_icon);
+      bookmarks_side_panel_icon)->SetVisible(false);
+
   bool other_folder_children_count =
       GetBookmarkMergedSurfaceService()->GetChildrenCount(
           BookmarkParentFolder::OtherFolder());
   if (other_folder_children_count) {
     menu->AppendSeparator();
     other_node_menu_separator_ = menu->GetSubmenu()->children().back().get();
+    other_node_menu_separator_->SetVisible(false);
   }
 }
