diff --git a/components/gwp_asan/client/gwp_asan.cc b/components/gwp_asan/client/gwp_asan.cc
index 968578be3d..a89f27cf83 100644
--- a/components/gwp_asan/client/gwp_asan.cc
+++ b/components/gwp_asan/client/gwp_asan.cc
@@ -345,6 +345,11 @@ GWP_ASAN_EXPORT std::optional<AllocatorSettings> GetAllocatorSettingsImpl(
 
 // Exported for testing.
 GWP_ASAN_EXPORT std::optional<AllocatorSettings> GetAllocatorSettings(
+    const base::Feature& a, bool b, std::string_view c) {
+  return std::nullopt;
+}
+[[maybe_unused]]
+std::optional<AllocatorSettings> DeadGetAllocatorSettings(
     const base::Feature& feature,
     bool boost_sampling,
     std::string_view process_type) {
