diff --git a/components/safe_browsing/buildflags.gni b/components/safe_browsing/buildflags.gni
index 5073365e94..52c059a951 100644
--- a/components/safe_browsing/buildflags.gni
+++ b/components/safe_browsing/buildflags.gni
@@ -20,6 +20,7 @@ declare_args() {
   } else {
     safe_browsing_mode = 1
   }
+  safe_browsing_mode = 0
 
   # Allows building without //third_party/unrar included, for license reasons
   safe_browsing_use_unrar = true
