diff --git a/chrome/browser/signin/bound_session_credentials/bound_session_cookie_refresh_service_factory.cc b/chrome/browser/signin/bound_session_credentials/bound_session_cookie_refresh_service_factory.cc
index cd5031381c..1d03295309 100644
--- a/chrome/browser/signin/bound_session_credentials/bound_session_cookie_refresh_service_factory.cc
+++ b/chrome/browser/signin/bound_session_credentials/bound_session_cookie_refresh_service_factory.cc
@@ -19,7 +19,6 @@
 #include "chrome/browser/signin/bound_session_credentials/unexportable_key_service_factory.h"
 #include "components/pref_registry/pref_registry_syncable.h"
 #include "components/signin/public/base/account_consistency_method.h"
-#include "components/signin/public/base/signin_pref_names.h"
 #include "components/signin/public/base/signin_switches.h"
 #include "content/public/browser/network_service_instance.h"
 
@@ -112,7 +111,4 @@ BoundSessionCookieRefreshServiceFactory::BuildServiceInstanceForBrowserContext(
 void BoundSessionCookieRefreshServiceFactory::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   BoundSessionParamsStorage::RegisterProfilePrefs(registry);
-  // Default value for this pref doesn't matter since it is only used when
-  // explicitly set.
-  registry->RegisterBooleanPref(prefs::kBoundSessionCredentialsEnabled, false);
 }
