diff --git a/chrome/windows_services/elevated_tracing_service/BUILD.gn b/chrome/windows_services/elevated_tracing_service/BUILD.gn
index bea12a8446..1014bcd74a 100644
--- a/chrome/windows_services/elevated_tracing_service/BUILD.gn
+++ b/chrome/windows_services/elevated_tracing_service/BUILD.gn
@@ -95,7 +95,6 @@ source_set("service_lib") {
   ]
   deps = [
     ":lib",
-    "//chrome/common/win:eventlog_provider",
     "//chrome/install_static:install_static_util",
     "//services/tracing/public/cpp",
   ]
